\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

% Bibliography
\usepackage[backend=biber,style=nature,backref=true]{biblatex}
\bibliography{/Users/jonwells/Documents/bibtex/Thesis-Introduction}
\let\cite\supercite


\begin{document}

\chapter{Introduction}

\section{What are protein complexes?}
% In their deeply impressive book, The Origin and Nature of Life on Earth \cite{Smith2016}, Smith and Morowitz describe two of the crucial steps in the origin of life as being the emergence of a single autocatalytic network at the core of today's metabolic network, and the appearance thereafter of large oligomers. Amongst these oligomers are proteins, which are now an integral part of both cellular and viral life.

% NOTE: were the first proteins enzymes then? Again, Gyorgy's stuff on protein

Perhaps the most convincing and powerful description of the Earth's biosphere is the one given by Smith and Morowitz in their deeply impressive book, The Origin and Nature of Life on Earth \cite{Smith2016}. In this work, they argue that rather than trying to define life at the level of the organism, it should instead be defined in aggregate - at the level of the biosphere. To be alive therefore is simply to be participating in the planet-wide energy flux that channels geothermal and solar energy through biological matter. At the heart of this energy flux, and thus life, lies a core metabolic network shared by all extant organisms. In the emergence of life, they argue, a crucial moment was the emergence of this autocatalytic network - the first step in `lifting chemistry off the rocks'. This was followed by the emergence of the oligomer world, amongst which proteins are a first-class member; indeed, an incredibly ancient set of such proteins now catalyse the reactions that comprise the universal core metabolic network.

% NOTE: Point out that the core metabolic network and the protein universe are deeply intertwined - read up about evolution of cofactors. Gyorgy's ligand binding paper would be good here, especially if you can hint at the link between diversification of protein species and the metabolic network. Symmetry breaking or something like that, in that perhaps multichain binding sites in homomers could lead to heteromers. THIS IS A GOOD PARAGRAPH, keep it but move till after you have defined homomers, heteromers etc. Why and how did protein complexes evolve, discuss Gyorgys work, then link this into Lynch's non-adaptive arguments.

% NOTE: Alternate first paragraph
% All cellular life on Earth is built on a foundation consisting of four core groups of biological macromolecules, namely: carbohyrdates, lipids, nucleic acids, and proteins. At the lowest level of abstraction, these groups can be differentiated by their chemical formulas, but for the sake of simplicity it is generally more convenient to separate them on the basis of their cellular function. Broadly speaking, carbohydrates and lipids act as stores of chemical energy, with the latter having additional important roles in signalling and membrane formation. Nucleic acids, which comprise DNA and RNA, are principally involved in information storage and transfer. Proteins - the subject of this thesis - are involved in essentially everything else.

% TODO Improve and tighten up section about complexity. Check your maths! If not 21 then need to correct this for all subsequent calcs too (and in footnote).
Proteins, such as those just mentioned, are biological macromolecules comprised of tens to thousands of amino acids, each amino acid linked by covalent peptide bonds between their terminal amine and carboxyl groups. There are 21 amino acids that are commonly used by biological organisms, meaning that a 200 residue sequence can be composed in \(20^{200}\) different ways.\footnote{Or in base 10: \(1.6\times10^{260}\). By way of comparison there are approximately \(4\times10^{80}\) atoms in the observable universe. Most of space is vacuum, after all.} To all indications, this level of complexity does not seem to be limited by biological constraints, and the Shannon entropy of the currently known sequence space does not differ markedly from shuffled sequences, particularly if you include viral proteins. However, unlike nucleic acid, which functions primarily as information storage, proteins are not defined by their sequence so much as their structure in three-dimensional space. When we factor in bond angles between residues the total possible sequence-conformational space rapidly becomes incalculable. Fortunately, we can usually ignore much of this detail, instead thinking of proteins as simple biological units, described in terms of their form and function.

% NOTE *Define* homomers and heteromers - introduce them more explicitly.
Functionally, all proteins can be defined in terms of their interaction with other molecules in the cell. In the case of enzymes, these are typically small metabolites; however, since most proteins are not enzymes, most biologically interesting interactions are in fact with other proteins. The majority of these protein-protein interactions are fleeting, arising as a result of intracellular crowding, and may or may not be functional. Many though are more long-lived, and produce stable protein complexes, the formation of which is generally essential to the proper function of the constituent protein subunits. To discuss these complexes further however, it is first necessary to make some definitions. When discussing the quaternary structure of a protein, there are three overarching classes that encompass all of quaternary structure space. A single protein chain that mostly functions without forming stable interactions with other proteins is known as a \textsc{monomer}. More common however, is the case where interactions occur between identical copies of a single protein species - the resulting complex is known as a \textsc{homomer}. Finally, \textsc{heteromers} are formed from groups of distinct, non-identical proteins. All classes are common, though there are intriguing differences in their relative frequencies in prokaryotes vs. eukaryotes, an observation which I will return to later.
% TODO Remove that last sentence if you don't return to it later.

% TODO Paragraph on what proteins do, and why they do things that can't be done by monomeric proteins. Point out that Lynch's argument applies to homomeric proteins, but it is probably less applicable to heteromers
If stable protein complexes are so common, then what (if any) evolutionary fitness benefit do they confer on the cell? A frequently overlooked explanation is the non-adaptive one, such as that proposed by Michael Lynch in 2013 \cite{Lynch2013}. In this important paper he first notes the extensive literature showing how homomeric interfaces can be formed or destroyed by simple point mutations in the protein of interest, and the fact that proteins are frequently found in different multimeric states in different organisms. Armed with these observations, he describes a simple model in which transitions between multimeric states are represented as a Markov process, the probabilities of which are dependent on the underlying mutation rate and selection pressure. The implications arising from this are that under neutral to modest selection pressure (in either direction), mutations that promote homomerisation of a given protein will arise as a consequence of genetic drift.
% NOTE does this lead to a gamma distribution? can I show that this fits the observed distribution of proteins in the PDB?

% TODO New paragraph, what are the main advantages - allostery, particularly in homomers, reference Monod allostery paper (1st citation in Lynch2013). For heteromers, it perhaps brings together different functional units, e.g metabolic pathways. A facile example could be actin and myosin, which obviously need something to lever against to generate force.
The fact that non-adaptive explanations are often sufficient to explain seemingly complex biological phenomena is an important one, and provides a crucial null hypothesis that should be addressed before turning to adaptive (Darwinian) explanations \cite{Koonin2016}. Similarly, exaptation, the process by which new functions are generated from redundant or no longer necessary parts \cite{Gould1979}. Undoubtedly, however all of the above contribute in some degrees to the the ubiquity of protein complexes.

There are numerous potentially adaptive benefits provided by the formation of protein complexes. For example, when considering the metabolic cost of synthesising proteins, it may be more effective to split a large protein into parts, so that errors in translation are restricted to smaller units - modularity is a key feature of most robust systems, whether biological or designed. This is pertinent since the error rates in gene expression are sufficiently high as to present a major challenge to the viability of life in general. The literature on this topic is extensive, but for a good review focussing on error control in translation see Gingold and Pilpel, 2011 \cite{Gingold2011}. For more spectacular examples of situations in which formation of complexes has enabled novel evolutionary solutions, one only has to look at ATP synthase, which, being a fully functional rotary motor, could not possibly exist in its current state were it formed from a single polypeptide chain.

% NOTE Read the Monod paper and use their definition, not some crackpot one you've made up. Also, consider moving the allostery paragraph to a slighrtly later section, since its kinda interesting in that if allostery was such a clear adaptive benefit then you might expect precise allosteric mechanisms to be conserved - they are not always! Can you cite Joe and Tee's deer project here?
A particularly widespread phenomena arising from the formation of protein complexes is that of allostery. The original definition of allostery given by Monod, Changeux and Jacob \cite{Monod1963} referred to modulation of protein activity by small molecules binding away from the active site, but this definition has since been extended to include cooperative effects between proteins. The classic example of this is of course haemoglobin, in which the binding of oxygen to one subunit increases the binding affinity of neighbouring subunits by the propagation of structural changes through the subunit binding interfaces \cite{Perutz1976}. Intriguingly however, though one might expect beneficial allosteric mechansims to be highly conserved, it turns out that the opposite is often the case; even haemoglobin, the textbook example, differs in the exact mechanism of allostery in different species. This begs the question: is some degree of allostery inevitable, in the same way as genetic drift will in many cases lead unavoidably to the formation of homomers?

% TODO What are the aims of this thesis? Why does assembly matter? Needs improving, is an important paragraph.
Thus far we have discussed protein complexes as fully formed entities. However, as we shall see over the course of this , the assembly process itself has great biological importance, and many phenomena can be better understood by taking it into account - the attenuation of protein levels observed in aneuploid cells (chapter 5) being a notable example. A central fact that underlies most of the work in this thesis is that the process by which protein complexes assemble is stochastic. However, whilst the binding of any two subunits is random - a product of Brownian motion and binding affinity - in aggregate, the assembly of protein complexes proceeds via non-random, ordered pathways \cite{Marsh2013}. This fact has profound implications for the behaviour of the cell.

\section{A brief overview of research on protein complexes}
% NOTE: does Molecular biology need capitalisation?
The tendency of proteins to form complexes and the functional implications of this have been understood since the earliest days of research on Molecular biology. Though it is not clear who was the first to explicitly note the existence of protein complexes, it seems likely that the realisation arose in tandem with investigations into the nature of viruses. In 1935, W. M. Stanley reported the isolation of `a crystalline material which has the properties of tobacco-mosaic virus', and demonstrated that this was predominantly composed of protein \cite{Stanley1935}. It is not obvious that he understood this material to be a protein complex in the sense that we would think of it, but certainly by 1955 the Tobacco mosaic virus capsid had been clearly described as a self-assembling homomer comprised of several thousand identical subunits \cite{Fraenkel-Conrat1955}.

% NOTE: scrap the bit about phage group - it has nothing to do with perutz, and watson has nothing to do with protein complexes.
This period in time marks a turning point for the field of biology. In the years directly after the second world war many physicists became interested in biology. In 1945, Max Delbr{\"u}ck founded the Phage Group, from which a number of now famous researchers emerged. Amongst these was James Watson, who in 1953, along with Francis Crick, Maurice Wilkins and Rosalind Franklin, discovered the structure of DNA, thus ushering in the modern era of biology \footnote{ Rosalind Franklin was also involved in pioneering work using crytallographic electron microscopy to investigate the structure of protein-DNA complexes. Sadly, she died in 1958 at the age of 37, before achieving the recognition she deserved, but this later work eventually led to her proteg{\'e},  Aaron Klug, winning the 1982 Nobel prize for Chemistry. It seems likely therefore, that, had she lived, Franklin would have been the recipient of two Nobels. Needless to say, this would have put her in rareified territory indeed.}. By the end of that decade, John Kendrew and Max Perutz had solved the structures of first Myoglobin, and shortly thereafter Haemoglobin \cite{Perutz} - the textbook protein complex.

Since these first post-war forays into the characterisation of protein complexes, technology has improved exponentially, and it is fair to say that during this period the field of structural biology has been one of the most fruitful in all of science. X-ray crystallography in particular has been a wild success, leading to no fewer than 14 Nobel Prizes since 1914. Of these, solving the structure of the ribosome - a huge complex consisting of dozens of protein and rRNA subunits - is perhaps the crowning achievement. However, whilst x-ray crystallography of most soluble monomeric proteins is now routine, there are still intractable difficulties for membrane proteins and large complexes. Recently, cryo-EM has seen a resurgence in popularity and effectiveness, led by combined improvements in electron detectors and image processing approaches. Crucially, since it does not rely on being able to crystallise the sample of interest, cryo-EM is able to fill in many of the gaps left by hard-to-crystallise proteins. It is particularly well-suited to looking at large protein complexes and is fast overtaking X-ray crystallography in this area.

% NOTE: When did mass spec appear - a return to non-structural methods?
Beyond structural methods, developments in mass spectrometry have led to a different, more nuanced understanding of the behaviour of proteins complexes and the wider proteome as a whole.

% NOTE: Plenty more to talk about with the advent of computational biology - work by Joe and colleagues, Lynch on non-adaptive hypotheses etc.

% NOTE: Conclude, and bridge to detailed discussion. It is on these non-structural methods that we will focus, but first an overview.

\section{Structural characterisation of protein complexes}

\subsection{X-ray crystallography}
% NOTE: For many years, the structural biology was synonoymous with X-ray crystallography, and certainly X-ray crystallography is the method that first made the field a reality.
X-ray crystallography was the first method to make the field of structural biology a reality, and brought together three key technologies, each useful in its own right. These technologies are: the ability to overexpress and purify proteins, the development of powerful sources of X-rays, and methods for resolving the resulting diffraction patterns produced when firing these X-rays at purified protein crystals. By and large, the ways in which X-ray crystallography can be used to determine protein structure are the same for monomeric proteins and those which form complexes. There are however some important differences and additional difficulties that are worth considering. Furthermore, although cryo-EM is poised to overtake X-ray crystallography as the method of choice for the solution of large heteromeric structures, there have been a number of exciting developments in the latter field that look set to ensure its future for many years to come. In the following section I will highlight of some of these advances, and attempt to give a summary of the current state of the field.

\subsubsection{Protein expression, purification, and crystallisation}
% NOTE: In this paper "Expression of protein complexes using multiple..." they talk about how there is no silver bullet for all protein complexes.
A generic protocol for the expression of a protein for cystallisation would involve transforming \textit{E. coli} with a plasmid containing your protein of interest, typically under the control of a strong, inducible promotor. Such a protocol works well for monomeric bacterial proteins, but expressing heteromeric protein complexes is often significantly more challenging. The key difficulty in the expression of heteromers lies in the production of sufficient quantities of pure sample, as in non-native host systems protein complex assembly is often inefficient or simply incomplete, making purification and subsequent crystallisation challenging. This problem - that of complexes being present in various oligomeric states - is one that also produces difficulties (albeit for different reasons) in cryo-EM projects.

At the simplest level, improvements in the cellular yield of bacterial heteromers can be achieved by considering the design of the expression vector, in light of the assembly pathway of the protein complex in question. As will be discussed in detail in chapter 2, the order of genes within protein complexes is under selection to match the assembly order of protein complexes \cite{Wells2016}. It has been demonstrated experimentally that taking this fact into account can markedly increase complex assembly efficiency, and that yields of heteromers in their fuly-assembled native state can be improved by using the native operon structure in expression vectors \cite{Shieh2015a, Poulsen2010}.

% NOTE: Need IMAC citation, review or otherwise.
When purifiying protein complexes there is a tradeoff between obtaining highly pure samples and ensuring that the intermolecular bonds between subunits are not disrupted. Though the diversity of methods for protein purification is bewilderingly high, in practice most methods suitable for protein complexes are variations on affinity purification. A standard, widely used protocol, mercifully summarised by Gr{\"a}slund et al. \cite{Graslund2008}, involves first generating a hexa-histidine tagged fusion protein. This `bait' protein is then expressed, preferably at native levels \footnote{Why do you not overexpress it? Think protein complex stoichiometry}, and the resulting cell extract is put through a column containing immobilised metal ions, resulting in the capture of the 6-His tagged protein, as well as anything else bound to it. Upon washing the column, the purified protein complex will be retained on the beads, and can be cleaved off the metal-coated beads in subsequent washes. Ideally, this would be ready for crystallisation, but in practice multiple washes and additional purification steps are generally required before the sample is sufficiently pure. Unfortunately, these additional steps will affect the final yield, and also risk disrupting more delicate interactions with peripheral subunits. Methods such as Dynamic Light Scattering are now frequently used to assess sample purity in a non-invasive manner.

% CITATIONS PLEASE!
The above provides a good starting point for designing a purification protocol, but in many cases it will be necessary to tailor the process to the protein complex of interest. Depending on the orientation of subunits within the structure for example, different subunits may make better or worse bait proteins, as will N- or C-terminal histidine tags. Similarly, some complexes may be disrupted by the presence of the metal ions, in which case other beads, e.g. those coated in calmodulin, may be more suitable. Although there has been some progress towards high-throughput expression and purification pipelines, much of this work still relies on the expertise of individual structural biologists and research technicians.

Surprisingly however, the crystallisation process is still the main bottleneck in X-ray crystallography, despite having been largely automated by the development of screening robots. There have however been some important methodological developments in the crystallisation of membrane proteins, which will also be useful for many membrane complexes. In particular, an exciting new method - X-ray solvent contrast modulation - has enabled for the first time visualisation of the interaction between bulk membrane phospholipids and embedded proteins \cite{Norimatsu2017}. However, this method still requires good quality crystals, and these are obtained through trial and error - beyond a few general rules of thumb we still do not have a good understanding of how different proteins will behave under varying crystallisation conditions.

% NOTE: shorten - just say how the slightly underwhelming performance of the JPSI highlight the difficulty of expression and crystallography - this is still a major bottleneck. Actually, there is some dev of high-throughput systems. e.g "Tandem recombineering by SLIC cloning and Cre-LoxP fusion to generate multigene expression constructs for protein complex research."

\subsubsection{Lasers, Electron and X-rays}
Once suitable crystals have been obtained, the major hurdle has been hurdled and image acquisition can begin. In contrast to earlier steps, enormous progress has been made in this domain since Lawrence Bragg first demonstrated that crystals diffract X-rays in XXXX. By far the most important development has been that of synchotron X-ray sources thus far. Synchotrons are able to produce X-rays at far higher intensities than traditional sources, and as such greatly reduce the time it takes to produce diffraction patterns. Additional properties of the X-rays can also be manipulated, for example narrowing the beamline in order to focus on the best quality region of the crystal, thus improving the resolution that can be achieved from the diffraction pattern.

XFEL - there are some incredibly cool looking papers on femtosecond crystallography - benefits are nano crystals, very short pulses, low radiation damage etc. This is the future. Short pulses means low damage, means no freezing means closer to native state? means protein complexes especially.

Synchotrons now widely used, enable higher intensity beams making sample collection much faster, enabling high-throughput crystallography. More recently hard X-ray free electron lasers have started being used, are about 10-13 orders of magnitude more intense than current synchotrons! The future of x-ray crystallography. Importance of cryocooling crystals for stability, largely demonstrated by Ada Yonath. Charge coupled devices being replaced by much faster pixel array detectors. For

% NOTE: IMPORTANT! need to get this more focussed on protein complexes. In particular, discuss the difficulty in distinguishing between crystal contacts and genuine interfaces - this leads to high eror rates in the PDB. Amongst different techniques available, which are most suitable and widely used for protein complexes?


\subsubsection{Figure, phase problem or something}

\subsubsection{Structure determination}
% NOTE: The structure of a protein in a crystal can be determined from its electron density function. This is a function of both the phases and amplitude of the x-ray diffraction maxima.
The crucial development that enabled the development of X-ray crystallography as a tool for determining the structure of proteins was Perutz's solution of the phase problem. Both the ability to crystallise proteins and the technological capability to visualise diffraction patterns had already been gained several years before, but it was the interpretation of these diffraction patterns that proved maddeningly difficult. Essentially, the phase problem is caused by the fact that, whilst diffraction patterns capture the amplitude of diffracted photons from a crystal (seen as the intensity of spots on the photograph), the phase of those photons is lost in collection. Unfortunately, it is the phases that carry the most information about the underlying structure, rather than the amplitude. This can be

Beyond isomorphous replacement, several other methods for circumventing the phase problem have now been developed. Closely related to ismorphous replacement is simply MIR, but a different class is that of SAD and MAD.

Software, the impact of CCP4, solving the phase problem etc. (Might be a nice place for a figure?) Use of polynuclear metal clusters for phasing large complexes \cite{Dauter2005}. There's a lot of different computational methods for phasing, is there any one that's best for complexes? Segue into cryoEM with some excited comments from leading crystallographers.
% NB concluding remarks in \cite{Shi2014}

\subsection{Cryo-electron microscopy}
A resurgence for an old technique in structural biology, now approaching crystallographic resolution, even for small, asymmetric proteins, e.g. \(\gamma\)-secretase. Will try and cover the recent advances that have made this possible.
\subsubsection{Electron detectors}
Important measure is "detective quantum efficiency", a measure of how signal to noise ratio is degraded by errors in imaging. Monolithic active pixel sensors are currently the most useful "direct electron detectors."
\subsubsection{Image processing and structure determination}
Loads of progress here and continuing all the time. Most widely used software is RELION, which makes use of a regularized likelihood (Bayesian) approach to image classification. That is, separating images of proteins/complexes into different classes based on rotational state and subunit heterogeneity etc.
\subsubsection{Figure, explain image classification problem?}
\subsubsection{room for improvement? Next steps?}


\section{Non-structural characterisation of protein complexes}
Don't need structural methods to look at complexes, can instead sometimes be characterised biochemically, and more useful, inferred from high-throughput proteomics data

\subsection{Yeast-two-hybrid and related methodologies}
mostly-very old methods, reimagined for 21st century. Most of their utility comes from simplicity and high-throughput modifications, won't spend too long on this.

\subsection{Affinity purification mass spectrometry}
Currently most powerful method for characterising interactomes, use some nice case-studies, in particular the way in which that 3D interactome paper \cite{Hein2015} combined AP-MS with quantitative mass spec (iBAQ) to get detailed stoichiometric information.

% NOTE Cross-linking

\subsubsection{Figure, AP-MS}

\section{Computational analysis of protein complexes}
\subsection{Inferring protein complexes from interaction networks}
spoke and matrix models, socio-affinity scoring. Read panorama paper. \cite{Wan2015}
\subsection{}
What can you do with the data, e.g. assembly order etc. Will probably spend some time discussing sequence-based methods, e.g. EVcouplings. Not especially well-suited to complexes due to size and corresponding computational requirements, but has been some interesting stuff looking at interfaces etc.

\section{Conclusion}
Wrap it up, try and find a nice way to lead into the main body of the thesis. In the chapter that follows, we will see a

These methods, particular structural ones, have enabled us to build an enormous reservoir of information about protein complexes, but it is now time to

\printbibliography

\end{document}
