\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

\bibliography{/Users/jonwells/Documents/bibtex/Thesis}
\graphicspath{{../figs/}}


\begin{document}

\chapter{Hawk proteins: A paralogous family of eukaryotic SMC-kleisin regulators} \label{chapter:hawks}

% NOTE: I think the important thing in this chapter is to get across that protein complexes, and the evolution of protein complexes, are essential drivers of new features. Tempting to speculate about the importance with regards to evolution of the nucleus.

\section{Introduction}
% NOTE: Possible continue by mentioning metazoan protein complexes?
The evolution of protein complexes differs from that of individual proteins in that functional adaptations, beneficial or otherwise, can be driven by gain and loss of subunits, in addition to the mechanisms that operate on individual proteins.

The SMC-kleisins (Structual Maintenance of Chromosomes) are an ancient family of protein complexes found in archaea, bacteria and eukaryotes. As the name suggests, these complexes have roles relating to maintenance of chromosomal architecture and DNA integrity, and operate at various stages of the cell cycle \cite{Nasmyth2009, Hirano2016}. Structurally, they are distinguished by their unusual tripartite ring formation, comprising two SMC arms that form a V-shaped dimer, linked by a largely disordered kleisin subunit (figure \ref{figure:smcs}). In eukaryotes there exist three main subfamilies of the SMC-kleisins: condensin, cohesin, and Smc5/6, whereas prokaryotes have (that we know of) MukBEF, MksBEF, and Smc/ScpAB.

\begin{figure}[h]
\fcapsideright
    {\caption[Eukaryotic members of the SMC-kleisin family of protein complexes]{\sffamily\textbf{Eukaryotic members of the SMC-kleisin family of protein complexes}\newline \small Structurally, they are distinguished by their unusual tripartite ring formation, comprising two SMC arms that form a V-shaped dimer, linked by a largely disorded kleisin subunit. In eukaryotes there exist three main subfamilies of the SMC-kleisins: condensin, cohesin, and Smc5/6, whereas prokaryotes have }\label{figure:smcs}}
    {\includegraphics{c2_fig_smcs}}
\end{figure}

Condensin and cohesin are responsible for two hallmark features of eukaryotic cell division, namely condensation of chromosomes and sister chromatid cohesion. Both complexes have been studied extensively, and condensin in particular is currently the subject of intense interest due to its central role in loop extrusion, a beautiful model that looks increasingly likely to be the correct mechanistic explanation for chromosome condensation \cite{Nasmyth2001a,Alipour2012,Goloborodko2016,Wang2017}. In contrast to condensin and cohesin, the structure and function of the Smc5/6 complex is less well-understood, but is known to be involved in DNA repair, specifically the resolution of recombination intermediates during mitosis and meiosis \cite{Ampatzidou2006,Farmer2011}. In practice however, the extensive literature on the family (reviewed recently by Frank Uhlmann \cite{Uhlmann2016}) indicates that there is probably a considerable degree of functional overlap between different members of the family, in both prokaryotes and eukaryotes.

% NOTE: The last few sentences of this and the entirety of the next paragraph are identical to the published article. Change this.
In addition to the SMC and kleisin subunits, numerous regulatory proteins are also associated with the complexes. One such group of regulators are the Kite proteins (\underline{K}leisin \underline{i}nteracting winged-helix \underline{t}andem \underline{e}lements), which form dimers that interact with the SMC-kleisin rings in bacteria, archaea, and the eukaryotic Smc5/6 complex. However, these are missing from condensin and cohesin, which instead interact with a number of proteins containing tandem HEAT (\underline{H}untingtin, \underline{E}F3, PP2\underline{A}, \underline{T}OR1) repeat motifs \cite{Andrade1995}. HEAT repeat proteins are a highly diverse family that are involved in a cell processes ranging from signalling (beta-catenin in the wnt pathway \cite{Morin1999}) to intracellular transport (clathrin adaptors \cite{McMahon2004} and karyopherins \cite{Chook2011}). Though it has been known for many years that a subset of these HEAT proteins are involved in the regulation of condensin and cohesin \cite{Neuwald2000}, neither their evolutionary history nor the extent of their coverage in the SMC family has been investigated. Building on the recent description of the Kite family, we asked whether this subset descends from a single ancestral HEAT repeat protein, and when this ancestor appeared.

% NOTE: Refer to methods section
Due to the nature of HEAT repeat proteins and repetitive sequences in general, this question presents non-trivial technical difficulties. Repetitive sequences can and often do diverge rapidly upon duplication \cite{Persi2016}; for example, the average sequence identity between mammalian HEAT repeat proteins and insect orthologues is just ~13\% \cite{Andrade2001c}. This makes accurate sequence alignment challenging, and therefore classical methods for homology detection fail on all but the most similar of these proteins. To tackle this problem we developed a computational approach based on extensive profile HMM searches and network clustering. Using this method, we were able to answer important questions about the origin of the HEAT protein regulators of condensin and cohesin, and thus we propose naming the group Hawks, i.e. HEAT proteins associated with kleisins.

\section{Results}
\subsection{Resolving evolutionary relationships between repeat proteins}
To detect potential paralogues of the candidate Hawk proteins, we used first used HHblits \cite{Remmert2011} to search the \textit{Saccharomyces cerevisiae} proteome (Uniprot ID UP000002311) for proteins with strong sequence similarity to the candidates. This immediately showed highly significant alignments between members of the Hawk group - alignments which were not detectable by less sensitive, but more widely used methods such as PSI-BLAST \cite{Altschul1997}. However, these searches also revealed significant similarities with other proteins from the HEAT family without known connection to SMC-kleisins. This raises an important issue, and motivated the development of a more rigorous method for assessing relationships.

By definition, repeat proteins contain multiple copies of a homologous sequence motif. Assuming for the sake of argument that each repeat has a roughly similar probability of producing a significant alignment, then the more repeats a protein has, the greater the probability of it appearing as a significant hit in searches with homologous repeat proteins. Thus, when searching for relatives of the Hawks, the presence of many proteins in the list of hits may not be indicative of close relationship, but rather a reflection of the fact that large HEAT proteins are more able to produce significant alignments.

In order to deal with this issue, we reasoned that, if two repeat proteins are closely related via a single common ancestor, then they should each produce high-ranking alignments with the other, regardless of which was used as the query sequence (alignments are ranked in a fashion familiar to users of BLAST) In contrast, high-ranking alignments between highly diverged repeat proteins and generic proteins containing many canonical copies of the repeat motif should only appear when the former is used as a query. This is because the diverged protein will be attracted to large repeat proteins in a way that isn't true to the same extent in reverse.

Thus, for every `seed' protein we queried, we performed reciprocal searches using all the proteins producing significant alignments with that seed. With the combined results from these searches, we generated a network, with each protein being represented by a node, and edges between them being weighted by the mean of two alignment ranks. Thus, edges with high weights are broadly indicative of closer ancestry between two nodes than edges with lower weights. Finally, we clustered these networks using the MCL algorithm \cite{VanDongen2000}, which in our case revealed numerous well defined sub-families of HEAT-related proteins. This method is summarised graphically in figure \ref{figure:clustering}A.

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c2_fig_clusters}}
    \caption[Evolutionary relationships between HEAT repeat proteins captured by network clustering]{\sffamily \textbf{Evolutionary relationships between HEAT repeat proteins captured by network clustering} \\ \small (A) Pipeline showing computational approach to generation of homology networks. (B) \textit{Saccharomyces pombe} network. Different colours represent different clusters obtained by MCL. To aid visualisation, only edges with a mean true positive probability $\leq$ 99.0\% are shown. Subgraphs an individual nodes that become disconnected upon application of this threshold are not shown, with the exception of Scc3; though a highly significant member of the hawk cluster, it appears to have diverged significantly further than other members.}
    \label{figure:clustering}
\end{figure}

To initiate our network, we selected a semi-arbitrary set of candidate Hawk proteins from budding yeast (\textit{Saccharomyces pombe}), including known HEAT repeat proteins and those that we suspected might be related. After carrying out Amongst a large number of diverse clusters, all HEAT repeat proteins known to interact with the $\alpha$-, $\beta$- and $\gamma$- kleisins \cite{Nasmyth2009, Hirano2016} form a distinct, and highly robust cluster (figure \ref{figure:clustering} B, see figure \ref{suppfigure:humannetwork} for human version). In humans and \textit{S. cerevisiae}, two clusters are formed, one containing SA/Scc3 proteins and a number of pseudogenes, and another containing all remaining condensin and cohesin hawks (figure \ref{c2_suppfig_network}). It should be noted here that the three species networks are not independent from each other, and are just complementary. This is a result of the fact that HHblits generates profile HMMs for each protein, which themselves are generated from multiple sequence alignments. Orthologous proteins from related species will therefore contain a considerable degree of overlap in the information content of their profile HMMs. However, replicating the networks in multiple species is useful as it allows us to look for Hawks that may have been gained or lost in different species.

In order to validate the clustering of the Hawks into a single group, we first carried out permutation tests by randomly shuffling the ranks of all alignments and regenerating the networks based on the newly assigned ranks. We repeated this process $10^{6}$ times, and each time observed whether or not a cluster containing all the Hawks, or all but the Scc3-related Hawks was obtained. The results from this showed the Hawk clustering to be highly significant in all three networks (p-value $< 1 \times 10^{-6}$), with minor reductions in significance being achieved by allowing other proteins to cluster along with the Hawks.

Though not expected (due to the way in which the significance, and thus rank of alignments are calculated) we also found no significant correlation between the length of alignment and its rank, indicating that clustering is not affected by the size of the proteins. We next sought to confirm that other clusters contained useful biological information. Several known protein families were recapitulated in individual clusters, for example the Maestro family, whose members contain a shared HEAT-like repeat motif, and the clathrin adaptor family \cite{Smith2003a, McMahon2004}. In addition, when we carried out GO-term analysis, we found that many clusters are significantly enriched for similar biological processes (table \ref{table:eukaryotehawks}). We are therefore confident that our method is robust, and that the clustering of hawks in a self-contained group is not an artefact.

\subsection{Nse5 and Nse6 are erroneously annotated as containing HEAT repeats}
Two proteins that are associated with the Smc5-6 complex have been previously reported to contain HEAT repeats. Our analysis fails to support these earlier findings, despite much effort to find evidence supporting the claims. Both Nse5 and Nse6 are markedly shorter than typical HEAT proteins, at 388 and 522 residues respectively in \textit{S. pombe}. In contrast the Hawks are typically over 1000 residues in length, and other HEAT proteins are often longer still. Though both proteins were included as seed sequences for the generation of our networks, neither clustered with the rest of the Hawks, or indeed with anything else in the case of Nse5. On the basis of this initially unexpected finding, we manually inspected the results from the Nse5-6 HHblits searches, and found that neither had alignments meeting reasonable significance thresholds with either the Hawks or indeed any other HEAT proteins.

%NOTE: On the plus side, you can go into some more detail about this interesting observation. Is it convergence? or coincidence? or saturation?

We then searched the literature for the primary HEAT annotations of these two proteins. To our knowledge, the first paper to suggest that Nse6 contained HEAT repeats was Pebernard et al., 2006 \cite{Pebernard2006}, which assigned the annotation on the basis of a result from a now obsolete structure prediction program, 3D-PSSM \cite{Kelley2000}. We therefore tried to replicate the finding with Phyre2 \cite{Kelley2015}, the successor program to 3D-PSSM, and found that the best matched fold was a pair of alpha-helices arising from an Integron cassette protein (PDB code: 3jrt). The Phyre2 score for this prediction was marked as being low confidence (56\%) and the template protein has no similarity to typical HEAT repeat proteins.

Curiously, we did see one hit with HHblits between Nse6 and Cnd3 in \textit{S. pombe} spanning a region approximately the length of 1 repeat (51 residues). However, this was ranked 8th out of a total of 10 alignments (typical proteins yield 10s-100s), with a true positive probability of 9.8\%, an expect value of 340 and p-value of 0.067. This alignment disappeared under different search parameters (both more and less sensitive), was not detected in Human, and budding yeast does not appear to have any orthologues of Nse6. Needless to say, this does not meet reasonable significance thresholds, but it is hard not to be intrigued. Whilst it could be indicative of either convergent or highly divergent evolution, it seems most likely that it that it is a spurious result arising from multiple testing.

With respect to the Nse5 annotation, we were unable to find any published evidence for it containing repeats in either the Pebernard paper or others, and are thus unsure as to where the annotation originally came from. Using HHRepID \cite{Biegert2008}, we then searched for any evidence of repeats, HEAT or otherwise, but found none under a range of different parameters and sensitivities. Having ruled out the possibility of Nse5-6 being bona-fide Hawks, an important observation arises, namely that Condensins and Cohesins have Hawks, but have lost the Kites. In contrast, Smc5-6 is alone amongst eukaryotic Smc-kleisin complexes in retaining Kites (the Nse1/3 subunits), but lacking Hawks.

For the avoidance of further confusion, we also note that \textit{S. cerevisiae}'s Kre29, though likely performing a similar role to S. pombe’s Nse6, shows no indication of being evolutionarily related. In humans however Slf2, though approximately twice the length, does produce significant alignments through its C-terminal end with Nse6 (true positive probability: 97.8\%, expect value: $2.81\times10^{-6}$). Finally, The cohesin regulator Scc4/MAU2 interacts with Scc2 and contains TPR repeats, which are structurally similar to HEATs. We initially considered the possibility that it might be related to the Hawks. Ultimately however, Scc4 (PDB code: 4XDN) is structurally very different from the Hawks, with much tighter curvature and a different alpha-helix layout. In terms of sequence, there is no apparent homology between Scc4 and any of the Hawks, and thus we feel confident in saying that Scc4 is unrelated to the Hawks.

%NOTE: This is the same as in the published article. Change
\subsection{Evolutionary and structural support for a common ancestor of Hawks}

Having demonstrated the close evolutionary relationship between hawks, we next looked to a possible origin for the family. Searches of sequence databases revealed orthologues in a set of species collectively accounting for all of the eukaryotic family tree \ref{table:eukaryotehawks}, strongly suggesting that the last eukaryotic common ancestor (LECA) contained at least one member of the family. It is known that several archaea species contain proteins with PBS lyase HEAT-like repeat domains \cite{Schlesner2009}, and we therefore decided to look for repeats in the recently discovered lokiarchaeaota, which is a member of the Asgard archaeal superphylum, members of which are amongst the closest relatives to the eukaryotes \cite{Spang2015,Zaremba-Niedzwiedzka2017}. Although several significant hits were found, based on sequence annotations and simple size comparisons it seems almost certain that these proteins are not directly related to the hawks. However, when we integrated them into our existing networks, they predominantly clustered with the clathrin adaptor family, and these (from sequence similarity alone) appear to be amongst the most closely related proteins to the hawks (figures \ref{figure:clustering}B, \ref{suppfigure:adaptin}).

% We next turn to a possible origin for the Hawk family. Orthologues were found in almost all eukaryote species we tested, collectively accounting for all major extant branches of the eukaryotic tree (Table S2). We next searched for related sequences in Lokiarchaeota, currently the closest known archaeal relative of the Last Eukaryotic Common Ancestor \cite{Spang2015} (LECA). Several lokiarchaeal HEAT repeat proteins produced significant alignments with Hawks, though based on simple size differences and gene annotations they do not appear to be functionally equivalent. However, we find that when integrated into our existing networks, the lokiarchaeal HEATs predominantly cluster with the Clathrin Adaptor proteins. On the basis of our findings and that of others, we note that these proteins share sequence and structural similarity with the Hawks \cite{Neuwald2000} (Fig.1B, Fig. S2B). These observations lead us to tentatively suggest that the ancestral Hawk protein derived from an ancient group of HEAT proteins related to the Clathrin Adaptor family, and that this occurred close to or even prior to the prokaryote-eukaryote split.

% NOTE: Giardia is lacking Scc1! no wonder it doesn't have Giardia
\begin{table}[h]
    \captionsetup{width=0.85\linewidth}
    \caption[Sample of eukaryotic supergroups with Hawk orthologues]{\sffamily \textbf{Sample of eukaryotic supergroups with Hawk orthologues} \\ \small Sequences were searched for manually using PSI-BLAST and HMMER \cite{Altschul1997,Finn2011}. \textit{G. lamblia} (4th from top) is an intriguing case since it possesses Smc orthologues, but apparently no kleisins. In this context, it is less surprising that it does not possess hawks, since all of those associated with cohesin (and probably of condensin) are known to bind to the kleisin subunit. Of course, this begs the question of how condensin or cohesin function in this organism?}
    \centering
    \onehalfspacing
    \begin{tabular}{l l l c}
    \hline
    Supergroup  & Group   &  Species &   Hawks\\[0.1cm]
    \hline
Archaeplastida &   Viridiplantae  &   \textit{Arabidopsis thaliana}     &  yes\\
Archaeplastida &   Rhodophyta     &   \textit{Galderia sulphuraria}     &  yes\\
Archaeplastida &   Glaucophyta    &   \textit{Cynanophora paradoxa}     &  no\\
Excavata       &   Diplomonadida  &   \textit{Giardia lamblia}          &  no\\
Excavata       &   Parabasalia    &   \textit{Trichomonas vaginalis}    &  yes\\
Excavata       &   Kinetoplastida &   \textit{Trypanosoma cruzi}        &  yes\\
Opisthokonta   &   Fungi          &   \textit{Saccharomyces cerevisae}  &  yes\\
Opisthokonta   &   Metazoa        &   \textit{Homo sapiens}             &  yes\\
Amoebozoa      &   Mycetozoa      &   \textit{Dictyolstelium discoideum}&  yes\\
Amoebozoa      &   Lobosa         &   \textit{Naegleria gruberi}        &  yes\\
SAR            &   Rhizharia      &   \textit{Plasmodiophora brassicae} &  yes\\
SAR            &   Stramenopiles  &   \textit{Aphanomyces invadans}     &  yes\\
SAR            &   Alveolata      &   \textit{Oxytricha trifallax}      &  yes\\
Unknown        &   Haptophyta     &   \textit{Emiliana huxleyi}         &  yes\\
Unknown        &   Cryptophyta    &   \textit{Guillardia theta}         &  yes\\
[0.1cm]
    \hline
    \end{tabular}
    \label{table:eukaryotehawks}
\end{table}

Finally, we used recently published Pds5B \cite{Ouyang2016} (PDB ID: 4PJU) and SA2 \cite{Hara2014} (5HDT) structures to further validate our findings from sequence analysis. Both structures share significant similarity in terms of overall morphology and their known interaction partners, Rad21 and Wapl. Furthermore, as the abridged version of this work was under peer review \cite{Wells2017} (See appendix I), the structures of Scc2 from \textit{Chaetonium thermophilum} \cite{Kikuchi2016} and \textit{Eremothecium gossypii} \cite{Chao2017} (5T8V and 5ME3) were published, and it too shares the characteristic S-shape of Pds5B and SA2, which differs noticeably from most other HEAT proteins.

Pds5B and orthologues in yeast contain a large, centrally located insertion or deletion that manifests as a large protrusion from the side of the structure. When we split the structure on either side of this alignment, we found that the two parts on either side aligned very well to the SA2 structure, with TM-scores significantly higher that expected for unrelated sequences (figure \ref{figure:smcstruc}A), an observation that was also noted by Lee et al. \cite{Lee2016}. Although a degree of skepticism is warranted due to the fact that tandem structural repeats are probably more likely to produce significant structural alignments than more traditional structures, the similarities here are convincing. Finally, we generated multiple sequence alignments of metazoan orthologoues of Pds5B and SA2 and used these to map sequence conservation onto the surfaces of the two structures, revealing broadly similar patterns of conservation that correspond to the binding regions of Rad21 and Wapl (figure \ref{figure:smcstruc} B)

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c2_fig_strucs}}
    \caption[Structural similarities between human hawks Pds5B and SA2]{\sffamily \textbf{Structural similarities between human hawks Pds5B and SA2} \\ \small (A) Structural alignment of Pds5B and SA2. Overall, the two structures align moderately well, but are disrupted by a large indel in the middle of Pds5B, which appears to be conserved across all Pds5 orthologues. When splitting the Pds5B structure into three pieces at the start and end of this indel, the two terminal pieces align much better to SA2. (B) Both structures bind the alpha-kleisin Rad21 and Wapl. They have similar patterns of conservation along the convex face of their structures, corresponding to the binding location of Rad21. Similarly, towards their N-termini they both have Wapl binding sites.}
    \label{figure:smcstruc}
\end{figure}

\clearpage

\section{Discussion}

The results of this work lead to three main conclusions. Firstly, we are confident that the hawk family is descended from a common ancestor. Although they have diverged too far from each other to build an accurate phylogenetic tree, their robust clustering into a single group, along with their structural and functional similarity strongly suggests that they are monophyletic. This explanation is more parsimonious than the alternative, in which convergent evolution lead to multiple HEAT repeat proteins being recruited to the Smc-kleisins independently. Nonetheless, it will be of great interest to see the structures of the hawks associated with the condensins, as this will provide a more complete picture we currently have, being restricted to cohesin.

Secondly, it seems highly likely that previous papers that had annotated Nse5 and Nse6 as containing HEAT repeats are incorrect on this point. We were unable to replicate the findings first reported by Pebernard et al., using a number of more powerful approaches. Since Nse5 and Nse6 are now known not to be hawks, this implies that Smc5-6 is unique in that it has retained the kite proteins, but has either lost or never had hawks. It therefore seems plausible that the gain of ancestral hawk proteins via successive duplications may have been the decisive event that initiated the evolution of present day condensins and cohesins.

% NOTE: finish by describing the protocoatomer hypothesis, refs in asgard paper.
Finally, the fact that various hawk homologues are found across all extant eukaryotic branches strongly suggests that they arose around the time of LECA, if not earlier. It does not seem likely that archaeal species possessed them, though the similarity between HEAT proteins in lokiarchaeota, the clathrin adaptors and the hawks is also indicative of their early evolution. Based on these three main conclusions, we tentatively suggest the model shown in figure \ref{figure:hawkmodel}. It is interesting to speculate about how these results fit into our broader understanding of the evolution of the nucleus.

\begin{figure}[h]
\fcapsideright
    {\caption[Eukaryotic members of the SMC-kleisin family of protein complexes]{\sffamily\textbf{Eukaryotic members of the SMC-kleisin family of protein complexes}\newline \small Structurally, they are distinguished by their unusual tripartite ring formation, comprising two SMC arms that form a V-shaped dimer, linked by a largely disorded kleisin subunit. In eukaryotes there exist three main subfamilies of the SMC-kleisins: condensin, cohesin, and Smc5/6, whereas prokaryotes have }\label{figure:hawkmodel}}
    {\includegraphics[width=0.6\textwidth]{c2_fig_hawkmodel}}
\end{figure}

% \printbibliography

\end{document}
