\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

\bibliography{/Users/jonwells/Documents/bibtex/Thesis}
\graphicspath{{../figs/}}


\begin{document}

\chapter{Operon gene order is optimised for ordered assembly of protein complexes}\label{chapter:operons}

\section{Introduction}
Work carried out over the course of the last decade has revealed that protein
complexes, both homomeric and heteromeric, assemble via ordered, energetically
favourable pathways. Assembly and disassembly pathways can be observed in vitro
using ESI-MS \cite{Hernandez2007}, and by looking at gene fusions it has been
shown that these pathways are evolutionarily conserved
\cite{Levy2008,Marsh2013}. These experiments are laborious and time-consuming,
but fortunately assembly order can be predicted with good accuracy
computationally if the structure of the complex is available; in most cases,
assembly order is simply determined by interface size, with larger interfaces
assembling earlier. This has been confirmed independently by a recent study
using a combination of MS, NMR spectroscopy and EM, and a second study showing
assembly order can be predicted by the number of coevolving residues between
different subunits \cite{Macek2017,Mallik2017}.

Given the central importance of protein complexes to most biological processes,
there is a strong pressure on the cell to ensure that they assemble correctly in
a timely and efficient manner. However, this process is inherently stochastic,
and takes place in a cellular environment in which the background concentration
of protein and other biological macromolecules is incredibly high
\cite{Swain2002}. For heteromeric protein complexes, this presents a serious
problem: how do protein subunits expressed from different genes find each other
in such a crowded environment? The cost of failure is not trivial, since
misassembly or non-specific interactions with other proteins can lead to the
formation of toxic aggregates. To give an example familiar to many of us, the
formation of such aggregates is implicated in a number of neurodegenerative
diseases \cite{Ross2004a}.

If the assembly pathways that are observed in vitro also occur in vivo, then we
might expect to see evidence of this in the regulatory systems possessed by the
cell. In bacteria and archaea, protein complexes are often encoded within
operons \cite{Mushegian1996,Dandekar1998}, where multiple genes are transcribed
onto a single polycistronic mRNA. This presents a possible opportunity for
enhancing the efficiency of protein complex assembly - if physically interacting
genes are closer together within operons, then this would increase the
likelihood of those subunits finding each other upon being translated. Thus, we
reasoned that there might be some correspondence between operon gene order and
assembly order. The results from this work show that this is indeed the case.

\section{Results}
\subsection{Encoding protein complexes within operons is likely to facilitate efficient assembly}
In order to test the hypothesis, we acquired a large set of heteromeric protein
complex structures from 70 bacterial and archaeal species. Describing these
protein complexes as lists of non-redundant gene/subunit pairs, we then mapped
the location of each gene pair in the genome of the species it came from. Out of
a total of 1079 gene pairs, 368 were encoded within the same transcriptional
unit - that is, translated from the same mRNA (figure
\ref{figure:operonabundance}A), with the remaining 711 being transcribed
separately.

\begin{figure}
    \makebox[\textwidth]{\includegraphics[width=0.88\textwidth]{c2_fig_abundance}}
    \caption[Encoding protein complexes within operons enhances assembly
    efficiency]{\sffamily \textbf{Encoding protein complexes within operons
    enhances assembly efficiency} \\ \small (A) Transcription, translation and
    assembly for heterodimeric subunits encoded by the same vs. different
    transcriptional units. (B) Differences in protein abundance correlations
    (Spearman's \(\rho\)) for gene pairs encoded within the same vs. different
    transcriptional units. The correlation between genes encoded within the same
    operon is significantly higher than for those in different transcriptional
    units (p-value = 0.002), as determined by randomly shuffling pairs between
    groups \(10^{5}\) times. (C) Protein complexes encoded within operons are
    significantly less abundant on average than those encoded in different
    transcriptional units, with significance determined using the Wilcoxon
    rank-sum test. Adapted from figure 1, Wells et al. \cite{Wells2016}}
    \label{figure:operonabundance}
\end{figure}

By transcribing genes in operons, any differences in expression levels due to
transcription rate are automatically removed, and instead variation in observed
protein abundances must be due to differences in translation or degradation
rates. It has been suggested therefore that one of the primary benefits of
encoding protein complexes in operons is the reduced stochasticity in gene
expression associated with operons \cite{Swain2004,Sneppen2010,Shieh2015a},
which is consistent with observations demonstrating that stoichiometry of most
protein complexes is tightly controlled in \textit{E. coli} \cite{Li2014b}. In
figure \ref{figure:operonabundance}B, we demonstrate that in \textit{E. coli},
as expected, protein abundances (obtained from PaxDB \cite{Wang2015}) of gene
pairs encoded in the same transcriptional unit are more closely correlated than
those not. The same trend was seen when combining data across all organisms for
which structures and operons were available, as well as when using absolute
protein synthesis rates using ribosome-profiling data \cite{Li2014b} (figure
\ref{suppfigure:operoncomparison}).

The likelihood of protein complex subunits randomly encountering each other in
the cell is greater for highly expressed complexes. Since operons will
necessarily lead to co-localisation of its freshly translated proteins, the
benefit to being operon-encoded should be particularly strong for lowly
expressed protein complexes \cite{Swain2002,Kovacs2009}. Supporting this
prediction, in figure \ref{figure:operonabundance}C, we show that there is a
highly significant tendency for operon-encoded subunits to be less abundant,
with an approximately order of magnitude difference in the median abundance of
the two groups.

\subsection{Adjacent genes within operons are more likely to physically
interact}
As suggested by the fact that lowly expressed protein complexes are more likely
to be encoded in operons, close proximity upon translation probably enhances the
efficiency of assembly (figure \ref{figure:intervening}A). Supporting this idea,
others have noted previously that adjacent genes are more likely to physically
interact \cite{Mushegian1996,Dandekar1998}, though these studies do not
explicitly focus on operon-encoded genes.

When comparing the number of adjacent genes that physically interact (208
interacting pairs out of 220 total) with the number of non-adjacent interacting
pairs (77 out of 148), there is a much higher tendency for adjacent genes to
share a physical interface (odds ratio = 15.8, p-value = \(5\times10^{-22}\),
Fisher's exact test). In figure \ref{figure:intervening}B we show that this
tendency extends beyond just adjacent genes, with the effect continuing for the
first two intervening genes. A highly similar trend is observed using pairwise
interaction data obtained from a large Y2H screen in \textit{E. coli}
\cite{Rajagopala2014} (figure \ref{figure:intervening}C); this is again
significant when comparing the likelihood of adjacent vs. non-adjacent gene pair
interaction (odds ratio = 2.7, p-value = 0.0002), despite the apparent weaker
ability of Y2H to detect interactions.

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c2_fig_proximity}}
    \caption[Adjacent genes within operons are more likely to encode physically
    interacting subunits]{\sffamily \textbf{Adjacent genes within operons are
    more likely to encode physically interacting subunits} \\ \small (A) Within
    a given complex, although all subunits interact indirectly, not all must
    necessarily share a physical interface. Within operons, the genes that code
    for these subunits can either be adjacent or non-adjacent. (B) Subunit pairs
    separated by number of intervening genes. Each bar is subdivided into those
    pairs that physically interact and those that don't. We define a physical
    interaction between two genes as their sharing an interface of > 200Ã…. (C)
    Analogous to B, but using binary interaction data obtained from Y2H screens
    \cite{Rajagopala2014}. Error bars are 68\% Wilson binomial confidence
    intervals. (D) Physical interfaces between adjacent genes are significantly
    larger on average than either non-adjacent genes or those encoded in
    different transcriptional units. P-values were calculated using Wilcoxon
    rank-sum tests. Adapted from figure 2, Wells et al. \cite{Wells2016}}
    \label{figure:intervening}
\end{figure}

Since the median length of operons in our dataset is fairly small (four genes),
the number of possible gene pairs is only slightly skewed towards non-adjacent:
a four gene operon has three adjacent and three non-adjacent pairs, but the
number of non-adjacent pairs increases rapidly as operons get larger. We
therefore reasoned that the apparent tendency of adjacent gene pairs to
physically interact might just be an artefact of the high representation of such
pairs in our dataset. To control for this possibility, we generated a null model
in which all of the genes were shuffled within their operons. We then compared
the number of observed gene pairs that were both adjacent and interacted with
the number expected under the null model, and repeated this process \(10^{5}\)
times (figure \ref{figure:interveningcontrol}A). In all cases the number of
observed interacting pairs was significantly greater than expected by chance. As
before, this is also significant, to a lesser degree, when using pairwise
interaction data generated by Y2H assays with \textit{E. coli} (figure
\ref{figure:interveningcontrol}B).

Since many of the possible gene pairs in our dataset arise from a particularly
large operon in \textit{Thermus thermophilus} (the \textit{nqo} operon, which
encodes respiratory chain complex 1), we repeated these analyses excluding this
operon, and considering this operon alone (figure \ref{suppfigure:nqooperon}).
These were both significant, indicating that the result was not due to features
unique to this operon. It also demonstrates that the trend holds within a single
operon, provided that it is sufficiently large.

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c2_fig_intcontrol}}
    \caption[Relationship between gene pair proximity and likelihood of physical
    interaction]{\sffamily \textbf{Relationship between gene pair proximity and
    likelihood of physical interaction} \\ \small (A) The number of physically
    interacting genes at different distances compared to a null model in which
    gene order in operons is shuffled. (B) shows the same result using binary
    interaction data from Y2H screens\cite{Rajagopala2014}. To assess the
    significance of the observed tendency for genes closer together within an
    operon to physically interact, permutation tests were performed by shuffling
    the order of genes within operons $10^{5}$ times. In each trial (i.e. a
    single shuffling of gene orders), the total number of intervening genes
    between physically interacting subunits was counted; to calculate the
    p-value, the number of occasions that this expected number of intervening
    genes (based on shuffled operons) was less than the true figure was divided
    by the number of trials. Adapted from figure S2, Wells et al.
    \cite{Wells2016} See also figure \ref{suppfigure:nqooperon}.}
    \label{figure:interveningcontrol}
\end{figure}

The observation that adjacent genes are more likely to physically interact could
have an effect on the interpretations of earlier work showing that gene fusion
events tend to preserved the order of assembly \cite{Marsh2013}, since adjacent
genes often undergo fusion events \cite{Pasek2006}. To test whether this earlier
observation was affected by our newer findings, we repeated the test for
assembly-conserving fusions using only adjacent genes, and found that there was
still a significant tendency for fusions to conserve assembly order (figure
\ref{suppfigure:fusion}).

In addition to the increased tendency of adjacent gene pairs within operons to
form a physical interface, we also observed that these interfaces are typically
larger than those formed by non-adjacent gene pairs. Figure
\ref{figure:intervening}D shows the distribution of interface sizes for those
proteins that physically interact within a protein complex. Whilst there is
significant overlap in the interface size distributions (since they are not
normalised between complexes), there is nonetheless a clear difference between
adjacent and non-adjacent gene pairs, regardless of whether encoded within the
same operon or not. This is surprising, and hints at a relationship between gene
order and assembly order, since we know that larger interfaces tend to form
earlier than smaller ones \cite{Levy2008,Marsh2013,Macek2017}.

\subsection{Operon gene order closely matches order of assembly}
Building on this finding, we next considered whether or not there could be a
correlation between the assembly order of protein complexes and their gene order
within operons. Thus far, we have demonstrated that there is a spatial
relationship between gene order and protein complex assembly, in that adjacent
genes are more likely to physically interact, and form larger interfaces when
they do so. However, since the assembly of a protein complex does not occur
instantaneously, there is also a temporal aspect of the problem to be
considered. Within bacterial and archaeal systems, there are two factors that
impose a temporal order on the expression of genes encoded within operons. The
first of these is the phenomenon of co-transcriptional translation, in which
ribosomes begin translating nascent mRNA as it is being transcribed
\cite{Byrne1964,Gowrishankar2004,Kohler2017}. The second is translational
coupling, in which translating ribosomes proceed directly from one gene to the
next, without being released from the mRNA \cite{Oppenheim1980,Levin-Karp2013}.

Both of these have the effect, initially at least, of ensuring that genes
towards the 5' end of the mRNA transcript will be translated before those at the
3' end. Thus, if protein complex subunits that tend to assemble earlier were
encoded at the start of operons, then this would likely increase the efficiency
of assembly. There are three different ways in which a given pair of adjacent
genes can assemble once translated. If the two proteins form a heteromeric
interface, then assembly is simultaneous and gene order is interchangeable for
that pair without affecting assembly efficiency. Alternatively, if the first
interface to form is homomeric, then gene order does have an effect - either
gene order can match assembly order, in which case the first gene to be
translated will also form an interface first, or it can be different from
assembly order. In figure \ref{figure:operonassembly}A we demonstrate these
three different scenarios.

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c2_fig_order}}
    \caption[Operon gene order reflects protein complex assembly
    order]{\sffamily \textbf{Operon gene order reflects protein complex assembly
    order} \\ \small (A) Three possible scenarios for the relationship between
    gene order and assembly order for a single gene pair. (B) Conservation of
    gene order for the three possible relationships described in A. Error bars
    are Wilson binomial confidence intervals and p-values were calculated with
    Fisher's exact test. (C) Assembly order matches gene order in 79.2\% of gene
    pairs whose order is evolutionarily conserved. P-value was calculated using
    the binomial test. Adapted from figure 3, Wells et al. \cite{Wells2016}}
    \label{figure:operonassembly}
\end{figure}

To investigate the potential relationship between gene order and assembly order,
we predicted assembly pathways for all of the operon-encoded heteromeric gene
pairs in our dataset, then separated all of the 220 adjacent pairs into the
three categories described above. We then considered the tendency of gene order
in each of these three groups to be evolutionarily conserved. As shown in figure
\ref{figure:operonassembly}B, there is a significant tendency for gene order to
be conserved in cases where it matches assembly order. This suggests that gene
order is constrained by the requirement that it not interfere with protein
complex assembly.

Considering the 72 pairs where gene order is evolutionarily conserved and
assembly of one subunit happens before the other, we found that there was a
striking correspondence between gene order and assembly order (figure
\ref{figure:operonassembly} C). In 57 out of 72 pairs gene order matched
assembly order (79\%, p-value = \(7 \times 10^{-7}\), binomial test), compared
to just 10 out of 29 cases (34\%) where gene order was not conserved. Thus,
selection for ordered protein complex assembly appears to be a major driver of
gene order in prokaryotic operons.

The likelihood of a physical interaction between two protein decreases as the
number of intervening genes between them on an operon increases (figure
\ref{figure:intervening}). We see a similar trend when looking at the
relationship between gene and assembly order, with the two features becoming
less likely to match as the distance between the two genes on the operon
increases (figure \ref{suppfigure:nonadjcomparison}). This is unsurprising, as
once proteins have dispersed around the site of translation, the beneficial
effect of protein co-localisation arising from tightly controlled gene order
becomes lost.

A feature of operons that some studies have noted is for genes towards the start
of operons to be expressed at higher levels \cite{Nishizaki2007,Lim2011}
\footnote{If correct, the explanation for this given by Lim et al. is
interesting \cite{Lim2011}. As a result of coupling transcription and
translation, genes at the 5' end of an operon will be transcribed and available
to ribosomes before those at the 3' end. Since ribosome binding occurs almost as
soon as transcription is underway, 5' genes will be actively translated for
slightly longer than those at the distal end of the operon. However, to explain
the magnitude of the effect they saw, they inferred that the efficiency of
translation must be \textasciitilde sixfold greater during transcription than
after. A satisfying mechanistic explanation of this was not given at the time,
but the recent discovery of the expressome complex \cite{Kohler2017} suggests a
plausible answer.}. This leads to a possible alternative explanation for the
correspondence between gene order and assembly order. If there is some
requirement for earlier assembling proteins to be expressed at higher levels,
then this could produce the same observation for different reasons. To rule out
this hypothesis, we note that there is no relationship between assembly order
and protein abundance and that gene order is a better predictor of assembly
order than abundance (figure \ref{suppfigure:geneorder}). We do however see a
weak, though insignificant, trend for upstream genes within operons to be more
abundant (table \ref{table:abundgeneorder}).

\begin{table}[h]
    \captionsetup{width=0.9\textwidth}
    \caption[Relationship between gene order and abundance for adjacent
    heteromeric subunits]{\sffamily \textbf{Relationship between gene order and
    abundance for adjacent heteromeric subunits} \\ \small Previous works have
    reported a significant tendency for upstream genes in operons to be more
    abundant \cite{Nishizaki2007,Lim2011}. Looking only at proteins that are
    members of protein complexes and encoded adjacent to each other within
    operons, we see a slight tendency for upstream genes to be more abundant.
    However, this is not significant (using the binomial test), and the trend is
    contradicted altogether when using ribosomal footprinting data
    \cite{Li2014b}. Adapted from table S1, Wells et al. \cite{Wells2016}}
    \centering
    \onehalfspacing
    \begin{tabular}{p{30mm} | p{29mm} p{29mm} p{29mm}}
    \hline
    &  \textbf{Subunit encoded by upstream gene is more abundant (\textit{E.
        coli} only)}  & \textbf{Subunit encoded by upstream gene is more
        abundant (All species)} &  \textbf{Subunit encoded by upstream gene has
        a higher rate of protein synthesis}\\[0.1cm]
    \hline
    \raggedright \textbf{Gene order is evolutionarily conserved} & 26/47
        (55.3\%) & 36/67 (53.7\%) &  20/43 (46.5\%)\\
    \raggedright \textbf{Gene order is not evolutionarily conserved} & 5/10
        (50.0\%) & 10/16 (62.5\%) & 3/9 (33.3\%)\\
    \raggedright \textbf{Total} & 31/57 (54.5\%) & 46/83 (55.4\%) & 23/42
        (44.2\%)\\[0.1cm]
    \hline
    \end{tabular}
    \label{table:abundgeneorder}
\end{table}


\subsection{Gene order matters most for lowly expressed protein complexes}
Despite the majority of adjacent gene pairs having gene orders that correspond
with assembly order, a significant proportion do not (\textasciitilde 25\% of
pairs, disregarding whether or not gene order is conserved). What might account
for the lack of correspondence in these cases? One possibility is that other
constraints on gene order override protein complex assembly in these cases.
Other studies have noted an analogous trend for gene order to match the order of
metabolic pathways \cite{Zaslaver2004,Kovacs2009}, so it is possible that
similar biological phenomena could be influencing our results. However, analysis
of gene ontology (GO) terms \cite{Huntley2015} did not reveal anything promising
(figure \ref{suppfigure:operongo}).

A more plausible explanation stems from our earlier observation that encoding
subunits within operons is more common for lowly expressed protein complexes. In
the immediate local environment of the active polyribosome, the concentration of
interacting subunits will be high, but will drop off rapidly as the protein
diffuses away from the site of translation. For proteins that are expressed at
high levels, precise control of gene order might therefore be less important,
since subunits still have a good chance of finding each other away from the site
of translation. In figure \ref{figure:operonexpression}, we show that in
\textit{E. coli} this does indeed seem to be the case, indicating that the
minority of gene pairs in which gene order does not correspond to assembly order
can mostly be explained by their high abundance. As for comparisons of
operon-encoded and non-encoded complexes (figures \ref{figure:operonabundance},
\ref{suppfigure:operoncomparison}), we see the same trend when considering all
organisms in our dataset, and also when using absolute protein synthesis rates
obtained from ribosomal profiling data (figure
\ref{suppfigure:operonexpression}).

\begin{figure}[h]
\fcapsideright
    {\caption[Gene pairs whose assembly order does not match gene order are
    highly expressed]{\sffamily\textbf{Gene pairs whose assembly order does not
    match gene order are highly expressed}\newline \small P-values were
    calculated with Wilcoxon rank-sum tests. Adapted from figure 4, Wells et al.
    \cite{Wells2016} See also figure
    \ref{suppfigure:operonexpression}.}\label{figure:operonexpression}}
    {\includegraphics[width=0.66\textwidth]{c2_fig_expression}}
\end{figure}

\section{Discussion}
These findings have a number of important consequences. From a technical
standpoint, they demonstrate that computational predictions of assembly order
are largely accurate and biologically meaningful. Together with earlier studies
on the evolutionary conservation of assembly pathways \cite{Levy2008,Marsh2013},
it is now fairly clear that in vitro pathways are similar to those in vivo - at
the very least in bacteria. In eukaryotes it is known that chaperones play an
extensive role in the assembly of complexes \cite{Ellis2006}, and proteins that
rely on these extensively probably deviate from easily predictable assembly
pathways. In archaea on the other hand, we simply don't know enough about them
to make confident assertions about the regulatory mechanisms they use to aid
assembly of complexes. For one thing, we have substantially less structural and
sequence information available for them \cite{Mukherjee2017}, and for another,
it is known that the structure of their operons differs from the canonical
bacterial template \cite{Koide2009}.

An important assumption in this work is that expression control is primarily at
the level of translation, with the mRNA-level expression of protein complex
subunits being relatively constant within operons. In \textit{E. coli}, which
accounts for most of the data in this study, a substantial number of operons can
produce alternative transcripts, thanks to the presence of internal promoters
and terminators within operons \cite{Conway2014}. In this work we have used the
DOOR 2.0 database of prokaryotic operons, which distinguishes between operons
and transcriptional units, i.e. a single mRNA output from an operon, and have
used the latter in all analyses. Nonetheless, it is possible that some of the
subtleties arising from overlapping transcriptional units within operons have
been missed in this work.

Bearing these caveats in mind, we can nonetheless infer some interesting
biological details about the assembly process. For example, the fact that there
should be such a strong correspondence between gene order and assembly order for
lowly expressed complexes implies that assembly must be taking place very close
to the site of translation. This is because once proteins begin to diffuse away
from the polyribosome, any knowledge about the order in which they were
translated is rapidly lost. In the case of less abundant protein complexes, the
effect of diffusion away from each other would be particularly strong, and as a
result the selective benefit they get from minimising the spatial and temporal
distance between interacting subunits is probably larger than for more abundant
proteins.

Taking this argument further, it seems likely that in many cases operon-encoded
complexes assemble co-translationally. In the specific case of the
\textit{Vibrio harveyi} heterodimeric luciferase complex, this has been
demonstrated experimentally \cite{Shieh2015a}, and there is considerable
evidence pointing to this being a common occurrence on a wider scale, not only
in archaea and bacteria but also eukaryotes
\cite{Duncan2011,Wells2015,Natan2017}.

These results demonstrate that the pressure to assemble protein complexes
quickly and efficiently has been a major constraint on the evolution of gene
order in bacteria. However, eukaryotes are enormously different from bacteria
and archaea in ways that prevent such a system as the one described here from
being possible. Most obviously, the majority of eukaryotic species do not
possess operons, and due to the existence of the nucleus, none couple
transcription and translation. Indeed, the existence of operons appears to be
largely decided by genome size and complexity, which on average is much greater
in eukaryotes than prokaryotes \cite{Nunez2013}. And yet, given the increased
size of most eukaryotic cells, along with the diversity of intracellular
organelles and compartments, it seems that the need for regulatory systems that
facilitate protein complex assembly should be at least equal to that of
prokaryotes, if not greater. The following chapters discuss a biological
phenomenon that appears to be driven by such requirements.

% \printbibliography

\end{document}
