\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

\bibliography{/Users/jonwells/Documents/bibtex/Thesis}
\graphicspath{{../figs/}}


\begin{document}

\chapter{Hawk proteins: A paralogous family of eukaryotic SMC-kleisin regulators}

% NOTE: I think the important thing in this chapter is to get across that protein complexes, and the evolution of protein complexes, are essential drivers of new features. Tempting to speculate about the importance with regards to evolution of the nucleus.

\section{Introduction}
% NOTE: namely needs a comma? Also, structural maintenance of chromosomes or stability of mini chromosomes? see p599 of Nasmyth2005 megareview. Could include as a footnote? was apparently redefined here: Strunnikov, A., Hogan, E. & Koshland, D. SMC2, a Saccharomyces cerevisiae gene essential for chromosome segregation and condensation, defines a subgroup within the SMC family. Genes Dev. 9, 587–599 (1995). (noted in Uhlen review article). and cite the diff complexes please.
The SMC-kleisins (Structual Maintenance of Chromosomes) are an ancient family of protein complexes found in archaea, bacteria and eukaryotes. As the name suggests, these complexes have roles relating to maintenance of chromosomal architecture and DNA integrity, and operate at various stages of the cell cycle \cite{Nasmyth2009, Hirano2016}. Structurally, they are distinguished by their unusual tripartite ring formation, comprising two SMC arms that form a V-shaped dimer, linked by a largely disorded kleisin subunit (Fig. 1). In eukaryotes there exist three main subfamilies of the SMC-kleisins: condensin, cohesin, and Smc5/6, whereas prokaryotes have (that we know of) MukBEF, MksBEF, and Smc/ScpAB.

\begin{figure}[h]
\fcapsideright
    {\caption[Eukaryotic members of the SMC-kleisin family of protein complexes]{\sffamily\textbf{Eukaryotic members of the SMC-kleisin family of protein complexes}\newline \rmfamily Structurally, they are distinguished by their unusual tripartite ring formation, comprising two SMC arms that form a V-shaped dimer, linked by a largely disorded kleisin subunit (Fig. 1). In eukaryotes there exist three main subfamilies of the SMC-kleisins: condensin, cohesin, and Smc5/6, whereas prokaryotes have }}
    {\includegraphics{c2_fig1_smcs}}
\end{figure}

Condensin and cohesin are responsible for two hallmark features of eukaryotic cell division, namely condensation of chromosomes and sister chromatid cohesion. Both complexes have been studied extensively, and condensin in particular is currently the subject of intense interest due to its central role in loop extrusion, a beautiful model that looks increasingly likely to be the correct mechanistic explanation for chromosome condensation \cite{Nasmyth2001a,Alipour2012,Goloborodko2016,Wang2017}. In contrast to condensin and cohesin, the structure and function of the Smc5/6 complex is less well-understood, but is known to be involved in DNA repair, specifically the resolution of recombination intermediates during mitosis and meiosis \cite{Ampatzidou2006,Farmer2011}. In practice however, the extensive literature on the family (reviewed recently by Frank Uhlmann \cite{Uhlmann2016}) indicates that there is probably a considerable degree of functional overlap between different members of the family, in both prokaryotes and eukaryotes.

% NOTE: The last few sentences of this and the entirety of the next paragraph are identical to the published article. Change this.
In addition to the SMC and kleisin subunits, numerous regulatory proteins are also associated with the complexes. One such group of regulators are the Kite proteins (\underline{K}leisin \underline{i}nteracting winged-helix \underline{t}andem \underline{e}lements), which form dimers that interact with the SMC-kleisin rings in bacteria, archaea, and the eukaryotic Smc5/6 complex. However, these are missing from condensin and cohesin, which instead interact with a number of proteins containing tandem HEAT (\underline{H}untingtin, \underline{E}F3, PP2\underline{A}, \underline{T}OR1) repeat motifs \cite{Andrade1995}. HEAT repeat proteins are a highly diverse family, a small subset of which regulate cohesin and condensins in eukaryotes \cite{Hirano2016} (Fig.1A). Building on the recent description of the Kite family, we asked whether this subset descends from a common ancestral HEAT repeat protein within the larger family.

% NOTE: Refer to methods section
However, this question presents significant technical difficulties. Repetitive sequences can diverge rapidly upon duplication \cite{Persi2016}; indeed, the average sequence identity between mammalian HEAT repeat proteins and insect orthologues is just ~13\% \cite{Andrade2001c}. This makes accurate sequence alignment challenging, and classical methods for homology detection fail on all but the most similar of these proteins. To tackle this problem we developed a novel network-based approach. Briefly, this method utilises extensive profile HMM searches carried out using HHblits \cite{Remmert2011}. The results of these searches are used to generate a network; this is then clustered, revealing groups of paralogous proteins. Though computationally intensive, we believe this approach should be widely applicable in inferring relationships between other highly diverged groups of proteins, particularly in cases where traditional phylogenetic methods are not possible.

\section{Resolving evolutionary relationships between repeat proteins}
To detect potential paralogues of the candidate Hawk proteins, we used first used HHblits to search the Saccharomyces cerevisiae proteome (Uniprot ID UP000002311) for proteins with strong sequence similarity. This immediately showed highly significant alignments between members of the Hawk group which were not detectable by less sensitive, but more widely used methods such as PSI-BLAST \cite{Altschul1997}. However, these searches also revealed significant similarities with other, non-SMC HEAT proteins, raising the key issue that motivated the development of our method. Since by definition repetitive proteins contain multiple copies of a similar motif, large proteins containing many copies of the canonical motif (e.g. TOR1) are more able to produce significant alignments with other members of the repeat family than smaller, more diverged members. Thus, their presence in the list of hits when searching for relatives of the Hawks may not be indicative of close relationship, but rather a reflection of the fact that large HEAT proteins are simply easier to align to other HEAT proteins.

To tackle this issue, we reasoned that, if two repeat proteins are closely related via a single common ancestor, then they should each produce high-ranking alignments with the other, regardless of which was used as the query sequence (alignments are ranked in a fashion familiar to users of BLAST etc.) In contrast, high-ranking alignments between highly diverged repeat proteins and generic proteins containing many canonical copies of the repeat motif should only appear when the former is used as a query. Thus, for every “seed” protein we queried, we performed reciprocal searches using all the proteins producing significant alignments with that seed. With the combined results from these searches, we generated a network, with each protein being represented by a node, and edges between them being weighted by the mean of two alignment ranks. Thus, edges with high weights are broadly indicative of closer ancestry between two nodes than edges with lower weights. Finally, we clustered these networks using the MCL algorithm \cite{VanDongen2000}, which in our case revealed numerous well defined sub-families of HEAT-related proteins.

Applying this method to a semi-arbitrarily selected set of candidate Hawk proteins, we find that amongst a large number of diverse clusters identified in budding yeast, all HEAT repeat proteins known to interact with α-, β- and γ- kleisins \cite{Nasmyth2009, Hirano2016} form a distinct cluster  (Fig.1B). Similarly, in the human network two closely interacting clusters, one of SA/Scc3 proteins and SA pseudogenes and one containing the other cohesin and all of the condensin HEATs are formed (Fig.S1B-C, S2A). It should be noted here that the three species networks are complementary to each other, and not independent. This is a result of the fact that HHblits generates profile HMMs for each protein, which themselves are generated from multiple sequence alignments. Orthologous proteins from related species will therefore contain a considerable degree of overlap in their profile HMMs. Replicating the networks in multiple species allows us to look at the differences between Hawks that may have been gained or lost in different species.

In order to validate the clustering of the Hawks into a single group, we first carried out permutation tests by randomly shuffling the ranks of all alignments and regenerating the networks based on the newly assigned ranks. We repeated this process 106 times, and each time observed whether or not a cluster containing all the Hawks, or all but the Scc3-related Hawks was obtained. The results from this showed the Hawk clustering to be highly significant in all three networks (p-value < 1x10-6), with minor reductions in significance being achieved by allowing other proteins to cluster along with the Hawks.

Though not expected (due to the way in which the significance, and thus rank of alignments are calculated) we also found no significant correlation between the length of alignment and its rank, indicating that clustering should not be affected by the size of the proteins. We next sought to confirm that other clusters contained useful biological information. Several known protein families were recapitulated in individual clusters, for example the Maestro family, whose members contain a shared HEAT-like repeat motif, and the Clathrin Adaptor family \cite{Smith2003a, McMahon2004}. GO-term analysis of biological processes also demonstrated highly significant enrichment in several clusters (Supplementary table 1). We therefore confidently conclude that Hawks form a distinct sub-family within the larger group of HEAT repeat proteins.

An important conclusion concerning NIPBL/Scc2 - the cohesin DNA loader – stems from the observation that it is confidently included in the Hawk cluster. In contrast to the other Hawks, no Scc2-kleisin interaction has yet been described. Nevertheless, in budding yeast, Scc2 and the Scc1 kleisin have been shown to be interdependent for their presence on DNA \cite{Fernius2013} and we therefore suggest that NIPBL/Scc2 will prove to interact with cohesin via its kleisin subunit, like other Hawks. It is relevant in this regard that Scc2 from Chaetomium thermophilum binds robustly to Scc1 but neither to Smc1 nor Smc3 (Hongtao Yu, personal communication).

A second important observation is that our analysis fails to support previous assertions that the proteins Nse5 and Nse6 associated with the eukaryotic Smc5-6 holocomplex contain HEAT repeats. Both Nse5 and Nse6 are markedly shorter than typical HEAT proteins, at 388 and 522 residues respectively in S. pombe. In contrast the Hawks are typically over 1000 residues in length, and other HEAT proteins are often longer still. Though both proteins were included as seed sequences for the generation of our networks, neither clustered with the rest of the Hawks, or indeed with anything in the case of Nse5. On the basis of this initially unexpected finding, we manually inspected the results from the Nse5-6 HHblits searches, and found that neither had alignments meeting reasonable significance thresholds with either the Hawks or indeed any other HEAT proteins.

We then searched the literature for the primary HEAT annotations of these two proteins. To our knowledge, the first paper to suggest that Nse6 contained HEAT repeats was Pebernard et al., 2006 \cite{Pebernard2006}, which assigned the annotation on the basis of a result from a now obsolete structure prediction program, 3D-PSSM \cite{Kelley2000}. We therefore tried to replicate the finding with Phyre2 \cite{Kelley2015}, the successor program to 3D-PSSM, and found that the best matched fold was a pair of alpha-helices arising from an Integron cassette protein (PDB code: 3jrt). The Phyre2 score for this prediction was marked as being low confidence (56\%) and the template protein has no similarity to typical HEAT repeat proteins. As to the Nse5 annotation, we were unable to find any published evidence for it containing repeats in either the Pebernard paper or others, and are thus unsure as to where its HEAT annotation came from. Using HHRepID \cite{Biegert2008}, we then searched for any evidence of repeats, HEAT or otherwise, but found none under a range of different parameters and sensitivities. Having ruled out the possibility of Nse5-6 being bona-fide Hawks, an important observation arises, namely that Condensins and Cohesins have Hawks, but have lost the Kites. In contrast, Smc5-6 is alone amongst eukaryotic Smc-kleisin complexes in retaining Kites (the Nse1/3 subunits), but lacking Hawks.

For the avoidance of further confusion, we also note that S. cerevisiae’s Kre29, though likely performing a similar role to S. pombe’s Nse6, shows no indication of being evolutionarily related. In humans however Slf2, though approximately twice the length, does produce significant alignments through its C-terminal end with Nse6 (true positive probability: 97.8\%, expect value: 2.81x10-6). Finally, The cohesin regulator Scc4/MAU2 interacts with Scc2 and contains TPR repeats, which are structurally similar to HEATs. We initially considered the possibility that it might be related to the Hawks. Ultimately however, Scc4 (PDB code: 4XDN) is structurally very different from the Hawks for which structures exist, with much tighter curvature and a different alpha-helix layout. In terms of sequence, there is no apparent homology between Scc4 and any of the Hawks, and thus we feel confident in saying that Scc4 is unrelated to the Hawks.

We next turn to a possible origin for the Hawk family. Orthologues were found in almost all eukaryote species we tested, collectively accounting for all major extant branches of the eukaryotic tree (Table S2). We next searched for related sequences in Lokiarchaeota, currently the closest known archaeal relative of the Last Eukaryotic Common Ancestor \cite{Spang2015} (LECA) . Several lokiarchaeal HEAT repeat proteins produced significant alignments with Hawks, though based on simple size differences and gene annotations they do not appear to be functionally equivalent. However, we find that when integrated into our existing networks, the lokiarchaeal HEATs predominantly cluster with the Clathrin Adaptor proteins. On the basis of our findings and that of others, we note that these proteins share sequence and structural similarity with the Hawks \cite{Neuwald2000} (Fig.1B, Fig. S2B). These observations lead us to tentatively suggest that the ancestral Hawk protein derived from an ancient group of HEAT proteins related to the Clathrin Adaptor family, and that this occurred close to or even prior to the prokaryote-eukaryote split.

An independent test of our conclusion that Hawks derive from a common ancestor is provided by structural analysis of yeast subunits Pds5/Pds5B \cite{Hara2014} and Scc3/SA2 \cite{Ouyang2016}. These structures bear similarities in their overall S-like shape and kleisin-binding patterns. Intriguingly, low resolution cryo-electron microscopy images of Scc2 demonstrate the same overall S-shape \cite{Chao2015}. Supporting this further, an unpublished structure of Scc2 from C. thermophilum (Hongtao Yu, personal communication) shows that its C-terminal region has a very similar shape and structure to Pds5, albeit lacking the indel found in the latter. Pds5B and SA2 also align well, though disrupted by an indel \cite{Lee2016} (Fig1C, Fig.S2C). When this region was omitted, the alignment improved considerably (Fig.1C). Finally, SA2 and Pds5B display similar patterns of conservation along their spines (Fig.S2D). These similarities between SA2 and Pds5B are particularly striking since from sequence analysis SA2/Scc3 has the weakest links to the rest of the Hawk cluster.

\section{Discussion}
Based on our main conclusions - the strong clustering of Hawks, their deep conservation across eukaryotes, and their absence from Smc5-6 complexes - we propose a model for the Smc-kleisin complex in LECA (Fig.1D). According to our hypothesis, the ancestral Hawk protein was recruited to the complex very early in eukaryotic history. Successive duplications of this protein displaced the Kites, leading to the predecessor of modern condensins, containing two Hawk regulators (similar to budding yeast’s Ycs4 and Ycg1), and to the cohesins, with three (Pds5, Scc3 and Scc2). An outstanding question from this model is whether or not Smc5-6 gained and then lost Hawks, or whether its lack of Hawks indicates that it forms a branch distinct from the cohesins and condensins. In any case, our results show that the Smc-kleisins can be separated into two groups – those containing Kites, and those containing Hawks; of these, the Hawk-Smc-kleisins appear to be uniquely eukaryotic. A key question emerging is whether the replacement of Kites by Hawks was associated with the acquisition by Smc-kleisin complexes of novel functions unique to condensin and cohesin. One property shared by both is the ability to organize loops of chromatin fibres around an axial core \cite{Nasmyth2009, Hirano2016}. Cohesin, which contains three Hawks and whose Scc3/SA subunits seem to have diverged furthest, may have later acquired the ability to hold sister chromatids together and to be cleaved by separase.


\printbibliography

\end{document}
