\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

\bibliography{/Users/jonwells/Documents/bibtex/Thesis}
\graphicspath{{../figs/wip/}}


\begin{document}

\chapter{Autosomal dosage compensation in aneuploid cells} \label{chapter:aneuploidy}

\section{Introduction}

Cancer cells are renowned for their high levels of chromosomal instability and unusual, sometimes bizarre karyotypes. The state in which a cell has an abnormal number of chromosomes is known as aneuploidy, and occurs to degrees in all known cancers. However, it is common even in non-cancer cells, and is a pervasive feature in most eukaryotic organisms. Isolates of wild yeast strains for example have been found to harbour a variety of different karyotypes, \cite{Hose2015}. In humans, approximately 0.1\% of the population carry an extra copy of chromosome 21 \cite{Presson2013}, a karyotype that famously results in Down's syndrome. In most cases however, aneuploidies that have been acquired through the germ-line or early in development are lethal - highlighting this, a recent study of spontaneous miscarriages found that approximately 45\% were the result of aneuploidies \cite{Jia2015}, and the true figure is probably higher still, since studies in mice suggest that mosaic aneuploid embryos fail to develop much beyond gastrulation, and thus would often pass clinically undetected \cite{Lightfoot2006}.

% NOTE: in both yeast an mammalian cells.
Mechanistically, aneuploidy is the result of failure of chromosomes to separate properly during cell division, either through non-disjunction of sister chromatids or delays in the movement of chromosomes to opposite poles of the cell during anaphase \cite{Compton2011}. For a cell that gains a chromosome, the immediate effect is a doubling of the copy number of all the genes residing on it, thus leading to a significant increase in mRNA and protein production. However, a common feature noted in several studies is that a considerable number of these proteins are attenuated compared to their expected abundances. \cite{Stingele2012,Dephoure2014,Goncalves2017}.

This attenuation is generally thought to caused by post-translational degradation of excess protein complex subunits, since attenuated proteins are enriched in protein complexes, and their mRNA transcript abundances scale correctly with copy number. Strongly supporting this, the model presented in the previous chapter explaining non-exponential degradation of proteins also correctly predicts the attenuation of proteins in aneuploid cells. In this chapter, I will briefly elaborate on this idea and attempt to explain some of the similarities and differences between non-exponential degradation in wild-type cells and protein attenuation in aneuploid cells.

\clearpage

\section{Results}

% NOTE: Is it <= 0.6 or < 0.6?
\subsection{Attenuation of protein complex subunits is unique to heteromers}
First, to confirm that attenuation of protein complex subunits is a feature unique to heteromers, I made use of data from Dephoure et al. \cite{Dephoure2014}. This dataset describes the relative fold change in the abundance of 2,581 genes from disomic \textit{S. cerevisiae} strains. Using the same definition given by Dephoure et al., attenuated proteins are those satisfying:
\begin{displaymath}
    \log_{2} \left( \frac{disomic}{wildtype} \right) \leq 0.6
\end{displaymath}
This value ensures that the observed SILAC ratio is at least three standard deviations away from the expected mean value of 1.0 (equivalent to doubling abundance). Mapping this dataset onto structural data from the PDB confirmed earlier observations that attenuation of expression is more common for members of multi-subunit protein complexes (figure \ref{figure:aneuploidy_qstype}).

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c5_fig1_qsdsratio}}
    \caption[Attenuation of protein complexes is unique to heteromers]{\sffamily \textbf{Attenuation of protein complexes is unique to heteromers} \\ \small Members of protein complexes are significantly more likely to be attenuated upon doubling of gene copy number. However, this effect is almost entirely driven by heteromeric complexes, with disomic ratios that differ significantly from that seen in monomers (median log2 disomic ratios of 0.623 and 0.948 respectively, Wilcoxon rank sum test p-value = 4.315e-22). In contrast, homomers (median 0.942) behave much the same as monomers and the difference between them is not at all significant.}
    \label{figure:aneuploidy_qstype}
\end{figure}

Importantly, this observation is consistent with the model presented in chapter 4, in which non-exponential degradation is a consequence of excess protein complex subunits being degraded more rapidly than bound ones. Adapted to this situation, disomic ratio is dependent on the ratio of bound to unbound subunits. Duplicating a single subunit in a heteromer increases the unbound, unstable fraction of that protein, which is promptly degraded, leading to a lower disomic ratio. Proteins that are predominantly monomeric should not experience any systematic attenuation, since their degradation rate will not be affected by binding partners and should remain roughly constant. Likewise, subunits of homomeric complexes will not be attenuated since changes in gene copy number will not cause stoichiometric imbalances.

\subsection{Similarities and differences between wild-type subunit degradation and aneuploid attenuation}

Many of the features we see in ED and NED proteins reappear here, suggesting that they are different manifestations of the same underlying biological phenomena. For example, as complex size increases, the average disomic ratio of subunits decreases (figure \ref{figure:aneuploidusubs}), analogous to the enrichment of NED proteins in large complexes. As before, this is probably explained by the fact that larger proteins have a greater proportion of obligate subunits which are protected from degradation once bound.


% NOTE: Can you do some simple mathematical calculations to show that rates of subunit exchange will have a significant impact on attenuation. Markov model like from the last chapter, but with bidirectional movement between states representing subunit exchange. Then just show steady state probabilities.
In contrast, if one imagines a typical heterodimer, the observed attenuation for each subunit will be determined by the degree to which that subunit is buffered by its binding partner. This in turn is dependent on binding affinities, dissociation rates and starting concentrations. For example, if the starting concentration of A is much higher than B, then even doubling [B] will have little effect on the amount of unbound B, and thus it will appear to be non-attenuated.

\begin{figure}[h]
    \makebox[\textwidth]{
        \includegraphics[width=\textwidth]{c5_fig2_usubsdsratio}
        }
    \caption[Degree of attenuation increases with increasing complex size]{\sffamily \textbf{Degree of attenuation increases with increasing complex size} \\ \small The abundance of monomers and homomeric subunits is largely determined by gene copy number, with log\textsubscript{2} disomic ratios being approximately normally distributed about 1. Heteromeric subunits however become increasingly likely to be attenuated as the increasing number of unique subunits. “Significant” attenuation is defined here by a threshold value of < 0.6, highlighted by the dashed red line. Structural and non-structural datasets have been combined here.}
    \label{figure:aneuploidusubs}
\end{figure}

One observation raised in Dephoure et al. is that almost all complexes have at least one non-attenuated subunit. An interesting idea mooted to explain this is the possibility that complexes require at least some stable subunits to act as a scaffold for the rest of the complex. If this were the case, then we would expect to see non-attenuated proteins being amongst the first to assemble. However, if we compare the disomic ratio of those subunits that assemble first against those that assemble last (figure \ref{figure:aneuploidy_assembly}), we find a clear (albeit weakly significant when controlling for number of unique subunits) tendency for early-assembling subunits to be attenuated. This finding matches the observations showing that NED subunits tend to assemble earlier than ED subunits, and is incompatible with the idea of stable proteins acting as scaffolds for protein complex assembly.

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c5_fig3_assembly}}
    \caption[Subunits that bind late to the complex are less likely to be attenuated]{\sffamily \textbf{Subunits that bind late to the complex are less likely to be attenuated} \\ \small Red line indicates “attenuated” threshold. P-values are Wilcoxon rank sum tests indicating the difference in disomic ratio between the first and last subunits to assemble. “Mid” subunits are all those that are neither first nor last to assemble. The non-significant values are probably mostly due to the fact that the “first” and “last” subunit numbers are highly limited by the number of available PDB structures, which is relatively low for the larger complexes.}
    \label{figure:aneuploidy_assembly}
\end{figure}

An interesting difference between NED proteins and attenuated proteins is in their abundance. At first glance, one might expect to see a similar case as for the NED proteins, where within complexes NED proteins tend to be more abundant than ED. However, when we look at this for the disomic ratio data (figure \ref{figure:aneuploidy_abundance}), there is no significant difference between proteins whose abundance is attenuated and vice versa. On reflection, this makes sense - in wild-type cells, the more abundant protein will be degraded non-exponentially, whereas in aneuploid cells, the natural abundance of the protein is rendered insignificant compared to the effect of copy number changes.

\begin{figure}[h]
\fcapsideright
    {\caption[Log2 fold-change in subunit abundance vs. median subunit abundance]{\sffamily\textbf{Log2 fold-change in subunit abundance vs. median subunit abundance}\newline \small When calculating the fold change of subunit abundance relative to the median subunit abundance within a complex, there is no significant difference between attenuated and non-attenuated proteins. This is in contrast to NED vs. ED, in which the former tend to be relatively more abundant. Fold change was calculated as log2 (subunit abundance/median abundance).}\label{figure:aneuploidy_abundance}}
    {\includegraphics[width=0.5\textwidth]{c5_fig4_abundance}}
\end{figure}

\section{Discussion}

Supplementing the finding that NED predicts the response of proteins to aneuploidy, these analyses go further in explaining the connection between attenuation in aneuploid cells and non-exponential degradation in normal cells. Both sets of results suggest that the assembly of a complex is not centred around stable scaffold proteins, but rather a core set of unstable, highly-expressed subunits. The first subunits to assemble tend to be rapidly degraded, with significant stabilisation presumably occurring upon binding, as indicated by the NED data and again here. In contrast, the last subunits to bind tend to be comparatively stable in both bound and unbound states.

The clearest difference between attenuation and non-exponential degradation is in the fact that NED is the result of relatively slight differences in protein expression within complexes, whereas attenuation is the result of much larger increases in expression affecting all of the proteins on affected chromosomes. There are still important questions to be tackled in understanding the behaviour of aneuploid cells - for example, to what extent does protein aggregation occur in aneuploid cells, and what are the factors determining whether a protein is degraded or aggregates?

\end{document}
