\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

\bibliography{/Users/jonwells/Documents/bibtex/Thesis}
\graphicspath{{../figs/wip/}}


\begin{document}

\chapter{Methods}

\section{Introduction}
The methods provided in this chapter relate to work performed by myself, and I have not included information describing experimental work carried out by others - this can instead be found in the online versions of the published papers included in appendix \ref{appendix:publishedpapers}. As the majority of the results described in this thesis have been published elsewhere, for the sake of accuracy I have not made any changes to the text as it appears in the published documents unless strictly necessary. In sections where methods have been lightly modified (marked with an asterisk) or quoted verbatim (two asterisks), permission for reuse has been granted.

\section{Methods}

\subsection{Chapter \ref*{chapter:hawks}: Hawk proteins: A paralogous family of eukaryotic SMC-kleisin regulators}

\subsubsection{Construction of homology networks *}
Proteome fasta files for \textit{S. cerevisiae}, \textit{S. pombe} and \textit{H. sapiens} were downloaded from the UniProt reference proteomes databank \cite{Consortium2017} (04.2016) and HHSuite v.3.0.0 was compiled from source \cite{Soding2005,Remmert2011}. HHsuite databases were constructed as per the protocol described in the HH-suite manual (available at http://www.mpibpc.mpg.de/soeding or https://github.com/soedinglab/hh-suite), using the clustered uniprot20\_2016\_02 database. It should be noted that due to the fact that HHsuite databases are generated from large multiple sequence alignments for each protein, the resulting species databases are not independent. Orthologous proteins in each species will, by virtue of that fact, produce profile HMMs with significant overlap.

Seed sequences for putative members of the Hawk family were selected semi-arbitrarily for each species (see supp. data file 1). Each seed was searched against the uniprot20 database using hhblits \cite{Remmert2011} (local alignment, two iterations). Predicted secondary structure was added to each MSA/profile HMM using Psipred \cite{Jones1999}. The resulting profile HMMs were then searched against the relevant species-specific database using hhsearch (local alignment, single iteration, no pre-filter) to generate a list of at most 500 putative paralogues from each seed. In turn, each one of these sequences was subjected to the same procedure, producing a large set of nodes and edges, with nodes representing proteins and edges representing alignments between them, weighted by the rank of the alignment.

The resulting graph was filtered by removing edges arising from alignments with a length of less than 100 columns (accounting for the length of ~2 HEAT repeats), an expect-value of greater than 0.01 (thus controlling the false-discovery rate) or a true positive probability of less than 15\%. Edge weights were then normalised according to the following formula, such that the normalised rank $f(r)$ lies between 0.01 and 1.0, with 1.0 being the best possible mean rank and 0.01 the worst.

\begin{displaymath}
    f(r) = \frac{1}{1 + \frac{99(r – r_{min} )}{r_{max} - r_{min}}}, 1 \leq r \leq 500
\end{displaymath}

At this stage, each edge has a direction, pointing from the protein used as a query sequence to the returned paralogous protein. As such, a given pair of nodes can be connected by either one edge or two - the former only being possible if a protein appeared exclusively in the second round of searches and was therefore not queried itself. In order to make the graph undirected, all nodes with a degree of less than 2 were discarded and the remaining edges between each pair of nodes combined and weighted by the geometric mean of normalised alignment ranks. Since the geometric mean is always lower than the arithmetic mean, this avoids giving too much weight to results from proteins with very few significant alignments.

Finally, clustering was carried out using the mcl algorithm with an inflation parameter I = 2.5 for all networks \cite{VanDongen2000}. Initial network construction and parameter setting was performed on a fully-labelled \textit{S. cerevisiae} network, but \textit{S. pombe} and \textit{H. sapiens} replicates were performed on blinded graphs, with genes in each cluster only being revealed after all filtering and cluster parameters had been fixed. GO term enrichment analysis was carried out using the Cytoscape BiNGO app, with GO `Biological Process' annotations \cite{Maere2005}. P-values were generated using the hypergeometric test and corrected for false discovery rate using the Benjamini-Hochberg method \cite{Benjamini1995,Maere2005}.

\subsubsection{Homology network permutation tests **}
Assuming a null hypothesis under which alignment ranks contain no information about the relative likelihood of two proteins being related, a single control network was constructed for each species. This was generated from the observed network by randomising the edge weights between each pair of nodes. This was achieved by pre-filtering alignments as usual, but randomly assigning ranks. These were then normalised and averaged as for the observed network. Each random network was then clustered and each cluster tested for membership of Hawk proteins; specifically we ask: does there exist a cluster in the random graph containing exclusively those proteins from the largest Hawk cluster in the observed graph? This process was repeated 106 times for each species, and the resulting p-value calculated as the number of times the complete Hawk cluster was seen, divided by the number of trials.

% NOTE: Do you actually discuss lokiarchaeota searches?
\subsubsection{Searching for lokiarchaeota HEAT repeat sequences **}
13 Lokiarchaeota proteins containing HEAT repeats were downloaded from the UniProt database; 9 on the basis of UniProt sequence annotations and an additional 4 proteins, including 2 fragments, on the basis of HHsuite searches and manual inspection. These sequences were searched against our human HHsuite database, and the resulting human sequences searched back against the lokiarchaeota database. A sub-graph was built using the same parameters as for the main eukaryote networks, leaving exactly 10 archaeal proteins remaining after quality control. The resulting set of edges was concatenated onto the human network and re-clustered.

% NOTE: I don't think this is included in this one?
\subsubsection{Mapping of repeat domain boundaries **}
Sequences from \textit{S. cerevisiae} hawks and clathrin adaptors were used to generate multiple sequence alignments with HHblits. Multiple sequence alignments were generated with the uniprot20\_2016\_02 database. These alignments were subsequently passed to the HHRepID web server \newline (https://toolkit.tuebingen.mpg.de/hhrepid). The threshold p-value for assigning repeat domain families was kept at 0.01, and the threshold for suboptimal self-alignments was set to 0.1, also the default. The number of HHblits iterations was set to 0 since we had produced our own MSAs in the preceeding step. Repeat predictions were collected from the HHRepID results with alignment stringencies between 0.0 and 0.3, depending on which value produced highest confidence predictions.

\subsubsection{Structural alignments and conservation mapping **}
Structures for human Pds5B and SA-2 were downloaded from the PDB (5HDT \cite{Ouyang2016} and 4PJU \cite{Hara2014} respectively, 28.04.2016). Structures were aligned in PyMol using TM-align \cite{PyMol2016,Zhang2005}, both globally and locally by splitting SA-2 and Pds5B at residues L436 and Y462 respectively and realigning each half. Conservation mapping was performed using multiple sequence alignments generated as follows: For Pds5B and SA-2, 1000 metazoan sequences for each were retrieved from the NCBI non-redundant sequence database using blastp, then clustered to 90\% sequence identity with usearch \cite{Altschul1990,Edgar2010}. The remaining sequences were then aligned in forward and reverse directions with MAFFT, MUSCLE and GlProbs, with a final composite MSA being generated with MergeAlign \cite{Katoh2002,Edgar2004,Ye2015,Collingridge2012}. Finally, these were mapped onto the PDB structures in Chimera \cite{Pettersen2004}.

\subsubsection{Analysis of putative Nse5 and Nse6 HEATS *}
Specific searches for HEAT-containing Nse5 and Nse6 homologues were carried out with the same parameters as for the main network – hhblits with 2 iterations to generate profile HMMs, followed by hhsearch to find significant alignments in the three main species datasets. Kre29 was used in place of Nse6 for S. cerevisiae, and Slf2 for Human. Subsequent searches using hhblits/hhsearch were carried out with more iterations for the hhblits step – this increases sensitivity but at the cost of accuracy in determining relative rank of alignments. Additional searches were performed in a wider variety of species using the proteome datasets available on the HHSuite webserver. Next, HHRepID \cite{Biegert2008} was used to try and detect repeats within Nse5-6 themselves (as opposed to HEAT containing homologues). As before human Slf2 was also checked, as was Kre29. Iterations ranging from 3-8 were used to generate the profile HMMs, thus spanning a wide range of sensitivities.

Finally, a literature search was performed to try and identify the published evidence for the Nse5-6 HEAT annotations. On the basis of evidence for HEATs in Nse6 presented by Pebernard et al. \cite{Pebernard2006}, we attempted to replicate their finding using the structural prediction server 3D-PSSM, which is now obsolete \cite{Kelley2000}. Following this, we used the Phyre2 web server \cite{Kelley2015} with the Nse6 sequence (UniProt id - O13688) using default settings.
\end{document}
