\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

\bibliography{/Users/jonwells/Documents/bibtex/Thesis}
\graphicspath{{../figs/wip/}}


\begin{document}

\chapter{Operon gene order is optimised for ordered assembly of protein complexes}

\section{Introduction}
Work carried out over the course of the last decade has revealed that protein complexes, both homomeric and heteromeric, assemble via ordered, energetically favourable pathways. Assembly and disassembly pathways can be observed in vitro using electrospray MS \cite{Hernandez2007}, and by looking at gene fusions it has been shown that these pathways are evolutionarily conserved \cite{Levy2008,Marsh2013}. These experiments are laborious and time-consuming, but fortunately assembly order can be predicted with good accuracy computationally if the structure of the complex is available; in most cases, assembly order is simply determined by interface size, with larger interfaces assembling earlier. This has been confirmed independently by a recent study using a combination of MS, NMR spectroscopy and EM, and a second study showing that assembly order can be predicted by the number of coevolving residues between different subunits \cite{Macek2017,Mallik2017}.

Given the central importance of protein complexes to most biological processes, there is a strong pressure on the cell to ensure that they assemble correctly in a timely and efficient manner. However, this process is inherently stochastic, and takes place in a cellular environment in which the background concentration of protein and other biological macromolecules is incredibly high \cite{Swain2002}. For heteromeric protein complexes, this presents a serious problem: how do protein subunits expressed from different genes find each other in such a crowded environment? The cost of failure is not trivial, since mis-assembly or non-specific interactions with other proteins can lead to the formation of toxic aggregates.

If the assembly pathways that are observed in vitro also occur in vivo, then we might expect to see evidence of this in the regulatory systems possessed by the cell. In bacteria and archaea, protein complexes are often encoded within operons \cite{Mushegian1996,Dandekar1998}, where multiple genes are transcribed onto a single polycistronic mRNA. This presents a possible opportunity for enhancing the efficiency of protein complex assembly - if physically interacting genes are closer together within operons, then this would increase the likelihood of those two subunits finding each other upon being expressed. Thus, we reasoned that there might be some correspondence between operon gene order and assembly order. The results from this work show that this is very much the case.

\section{Results}
\subsection{Encoding protein complexes within operons is likely to facilitate efficient assembly}
In order to test this hypothesis, we acquired a large set of heteromeric protein complex structures from 70 bacterial and archaeal species. Describing these protein complexes as lists of non-redundant gene/subunit pairs, we then mapped the location of each gene pair in the genome of the species it came from. Out of a total of 1079 gene pairs, 368 were encoded within the same transcriptional unit - that is, translated from the same mRNA (fig. \ref{c3f1}A), with the remaining 711 being transcribed separately.

\begin{figure}
    \makebox[\textwidth]{\includegraphics[width=0.88\textwidth]{c3_fig1_abundance}}
    \caption[Encoding protein complexes within operons enhances assembly efficiency]{\sffamily \textbf{Encoding protein complexes within operons enhances assembly efficiency} \\ \small (A) Transcription, translation and assembly for heterodimers encoded by different the same vs. different transcriptional units.\\
    (B) Differences in protein abundance correlations (Spearman's \(\rho\)) for gene pairs encoded within the same vs. different transcriptional units. The correlation between genes encoded within the same operon is significantly higher than for those in different transcriptional units (p-value = 0.002), as determined by randomly shuffling pairs between groups \(10^{5}\) times.\\
    (C) Protein complexes encoded within operons are significantly less abundant on average than those encoded in different transcriptional units, with significance determined using the Wilcoxon rank-sum test.}
    \label{c3f1}
\end{figure}

By transcribing genes in operons, any differences in expression levels due to transcription rate are automatically removed, and instead variation in observed protein abundances must be due to differences in translation or degradation rates. It has been suggested therefore that one of the primary benefits of encoding protein complexes in operons is the reduced stochasticity in gene expression associated with operons \cite{Swain2004,Sneppen2010,Shieh2015a}, which is consistent with observations demonstrating that stoichiometry of most protein complexes is tightly controlled in \textit{E. coli} \cite{Li2014b}. In figure \ref{c3f1} B, we demonstrate that in \textit{E. coli}, as expected, protein abundances (obtained from PaxDB \cite{Wang2015}) of gene pairs encoded in the same transcriptional unit are more closely correlated than those not. The same trend was seen when combining data across all organisms for which structures and operons were available, as well as when using absolute protein synthesis rates using ribosome profiling data \cite{Li2014b} (figure S\ref{c3fs1}).

The likelihood of protein complex subunits randomly encountering each other in the cell is greater for highly expressed complexes. Since operons will necessarily lead to co-localisation of its freshly translated proteins, the benefit to being operon-encoded should be particularly strong for lowly expressed protein complexes \cite{Swain2002,Kovacs2009}. Supporting this prediction, in figure \ref{c3f1} C, we show that there is a highly significant tendency for operon-encoded subunits to be less abundant, with an approximately order of magnitude difference in the median abundance of the two groups.

\clearpage
\subsection{Adjacent genes within operons are more likely to physically interact}
% NOTE: Is the previously observed tendency of adjacent genes to interact a consequence of operons or independent of it?
As suggested by the fact that lowly expressed protein complexes are more likely to be encoded in operons, close proximity upon translation probably enhances the efficiency of assembly (figure \ref{c3f2} A). Others have noted previously that adjacent genes are more likely to physically interact when expressed \cite{}. When comparing the number of adjacent genes that physically interact (208 interacting pairs out of 220 total) with the number of non-adjacent interacting pairs (77 out of 148), there is a much higher tendency for adjacent genes to share a physical interface (odds ratio = 15.8, p-value = \(5\times10^{-22}\), Fisher's exact test). In figure \ref{c3f2} B we show that this tendency extends beyond just adjacent genes, with the effect continuing for the first one or two intervening genes. A highly similar trend is observed using pairwise interaction data obtained from a large Y2H screen in \textit{E. coli} \cite{Rajagopala2014} (figure \ref{c3f2} C), which is again significant when comparing the likelihood of adjacent vs. non-adjacent gene pair interaction (odds ratio = 2.7, p-value = 0.0002), despite the weaker ability of Y2H to detect interactions.

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c3_fig2_proximity}}
    \caption[Adjacent genes within operons are more likely to encode physically interacting subunits]{\sffamily \textbf{Adjacent genes within operons are more likely to encode physically interacting subunits} \\ \small (A) Within a given complex, although all subunits interact indirectly, not all must necessarily share a physical interface. Within operons, the genes that code for these subunits can either be adjacent or non-adjacent.\\
    (B) Subunit pairs separated by number of intervening genes. Each bar is subdivided into those pairs that physically interact and those that don't. We define a physical interaction between two genes as their sharing an interface of > 200Ã….\\
    (C) Analogous to C, but using binary interaction data obtained from Y2H screens \cite{Rajagopala2014}. Error bars are 68\% Wilson binomial confidence intervals.\\
    (D) Physical interfaces between adjacent genes are significantly larger on average than either non-adjacent genes or those encoded in different transcriptional units. P-values were calculated using Wilcoxon rank-sum tests.}
    \label{c3f2}
\end{figure}

Since the median length of operons in our dataset was fairly small (four genes) and therefore the number of possible gene pairs only slightly skewed towards non-adjacent (a four gene operon has three adjacent and three non-adjacent pairs), we reasoned that the apparent tendency of adjacent gene pairs to physically interact might just be an artefact of our dataset. To control for this possibility, we generated a null model in which all of the genes were shuffled within their operons. We then compared the number of observed gene pairs that were both adjacent and interacted with the expected number under the null model, and repeated this process \(10^{5}\) times (figure S\ref{c3fs2}). In all cases the number of observed interacting pairs was greater than expected, demonstrating that this trend is highly significant.

The observation that adjacent genes are more likely to physically interact could have an effect on the interpretations of earlier work showing that gene fusion events tend to preserved the order of assembly \cite{Marsh2013}, since adjacent genes are often undergo fusion events \cite{Pasek2006}. To test whether this earlier observation was affected by our newer finding, we repeated the test for assembly-conserving fusions using only adjacent genes, and found that there was still a significant tendency for fusions to conserve assembly order (figure S\ref{c3fs3}).

In addition to the increased tendency of adjacent gene pairs within operons to form a physical interface, we also observed that these interfaces are typically larger than those formed by non-adjacent gene pairs. Figure \ref{c3f2} D shows the distribution of interface sizes for just those proteins that physically interact within a protein complex. Whilst there is significant overlap since the sizes are not normalised between complexes, there is nonetheless a clear difference between adjacent and non-adjacent gene pairs, regardless of whether encoded within the same operon or not. This is surprising, and hints at a relationship between gene order and assembly order, since we know that larger interfaces tend to form earlier than smaller ones \cite{Levy2008,Marsh2013,Macek2017}.

\subsection{Operon gene order closely matches order of assembly}
Building on this finding, we next considered whether or not there could be a correlation between the assembly order of protein complexes and their gene order within operons. Thus far, we have demonstrated that there is a spatial relationship between gene order and protein complex assembly, in that adjacent genes are more likely to physically interact, and form larger interfaces when they do so. However, since the assembly of a protein complex does not occur instantaneously, there is also a temporal aspect of the problem to be considered. Within bacterial and archaeal systems, there are two factors that impose a temporal order on the expression of genes encoded within operons. The first of these is the phenomenon of co-transcriptional translation, in which ribosomes begin translating nascent mRNA as it is being transcribed \cite{Byrne1964,Gowrishankar2004,Kohler2017}. The second is translational coupling, in which translating ribosomes proceed directly from one gene to the next, without being released from the mRNA \cite{Oppenheim1980,Levin-Karp2013}.

Both of these have the effect, initially at least, of ensuring that genes towards the 5' end of the mRNA transcript will be translated before those at the 3' end. Thus, if protein complex subunits which tend to assemble earlier are encoded at the start of operons, then this would likely increase the efficiency of assembly. There are three different ways in which a given pair of adjacent genes can assemble once translated. If the two proteins form a heteromeric interface, then assembly is simultaneous and gene order is interchangeable for that pair without affecting assembly efficiency. Alternatively, if the first interface to form is homomeric, then gene order does have an effect - either gene order can match assembly order, in which case the first gene to be translated will also form an interface first, or it can be different from assembly order. In figure \ref{c3f3} A we demonstrate these three different scenarios.

\begin{figure}[h]
    \makebox[\textwidth]{\includegraphics[width=\textwidth]{c3_fig3_order}}
    \caption[Operon gene order reflects protein complex assembly order]{\sffamily \textbf{Operon gene order reflects protein complex assembly order} \\ \small (A) Three possible scenarios for the relationship between gene order and assembly order for a single gene pair.\\
    (B) Conservation of gene order for the three possible relationships described in A. Error bars are Wilson binomial confidence intervals and p-values were calculated with Fisher's exact test.\\
    (C) Assembly order matches gene order in 79.2\% of gene pairs whose order is evolutionarily conserved. P-value was calculated using the binomial test.}
    \label{c3f3}
\end{figure}

To investigate the potential relationship between gene order and assembly order, we predicted assembly pathways for all of the operon-encoded heteromeric gene pairs in our dataset, then separated all of the 220 adjacent pairs into the three categories described above. We then considered the tendency of gene order in each of these three groups to be evolutionarily conserved. As shown in figure \ref{c3f3} B, there is a significant tendency for gene order to be conserved in cases where it matches assembly order. This suggests that gene order is constrained by the requirement that it not interfere with protein complex assembly.

Considering the 72 pairs where gene order is evolutionarily conserved and assembly of one subunit happens before the other, we found that there was a striking correspondence between gene order and assembly order (figure \ref{c3f3} C). In 57 out of 72 pairs gene order matched assembly order (79\%, p-value = \(7 \times 10^{-7}\), binomial test), compared to just 10 out of 29 cases (34\%) where gene order was not conserved. Thus, selection for ordered protein complex assembly appears to be a major driver of gene order in prokaryotic operons.

The likelihood of a physical interaction between two protein decreases as the number of intervening genes between them on an operon increases (figure \ref{c3f2}). We see a similar trend when looking at the relationship between gene and assembly order, with the two features becoming less likely to match as the distance between the two genes on the operon increases (figure S\ref{c3fs4}). This is unsurprising, as once proteins have dispersed around the site of translation, the beneficial effect of protein co-localisation arising from tightly controlled gene order becomes lost.

A feature of operons that some studies have noted is for genes towards the start of operons to be expressed at higher levels \cite{Nishizaki2007,Lim2011} \footnote{If correct, the explanation for this given by Lim et al. is interesting \cite{Lim2011}. As a result of coupling transcription and translation, genes at the 5' end of an operon will be transcribed and available to ribosomes before those at the 3' end. Since ribosome binding occurs almost as soon as transcription is underway, 5' genes will be actively translated for slightly longer than those at the distal end of the operon. However, to explain the magnitude of the effect they saw, they inferred that the efficiency of translation must be \textasciitilde sixfold greater during transcription than after. A satisfying mechanistic explanation of this was not given at the time, but the recent discovery of the expressome complex \cite{Kohler2017} hints at an attractive answer to the problem}. This leads to a possible alternative explanation for the correspondence between gene order and assembly order. If there is some requirement for earlier assembling proteins to be expressed at higher levels, then this could produce the same observation for different reasons. To rule out this hypothesis, we note that there is no relationship between assembly order and protein abundance and that gene order is a better predictor of assembly order than abundance (figure S\ref{c3fs5}). We do however see a weak (though insignificant) trend for upstream genes within operons to be more abundant (\textbf{MISSING TABLE}).

\subsection{Gene order matters most for lowly expressed protein complexes}
Despite the majority of adjacent gene pairs having gene orders that correspond with assembly order, a significant fraction do not (\textasciitilde 25\% of pairs, disregarding whether or not gene order is conserved). What might account for the lack of correspondence in these cases? One possibility is that other constraints on gene order override protein complex assembly in these cases. Other studies have noted an analogous trend for gene order to match the order of metabolic pathways \cite{Zaslaver2004,Kovacs2009}, so it is possible that similar biological phenomena could be influencing our results. However, analysis of gene ontology terms \cite{Huntley2015} did not reveal anything promising (figure S\ref{c3fs6}).

% NOTE Fick's law, maybe put something in the appendix?
A more plausible explanation stems from our earlier observation that encoding subunits within operons is more common for lowly expressed protein complexes. In the immediate local environment of the active polyribosome, the concentration of interacting subunits will be high, but will drop off rapidly as the protein diffuses away from the site of translation. For proteins that are expressed at high levels, precise control of gene order might therefore be less important, since subunits still have a good chance of finding each other away from the site of translation. In figure \ref{c3f4}, we show that in \textit{E. coli} this does indeed seem to be the case, indicating that the minority of gene pairs in which gene order does not correspond to assembly order can mostly be explained by their high abundance. As for comparisons of operon-encoded and non-encoded complexes (figures \ref{c3f1}, S\ref{c3fs1}), we see the same trend when considering all organisms in our dataset, and also when using absolute protein synthesis rates obtained from ribosomal profiling data (figure \ref{c3fs7}).

\begin{figure}[h]
\fcapsideright
    {\caption[Gene pairs whose assembly order does not match gene order are highly expressed]{\sffamily\textbf{Gene pairs whose assembly order does not match gene order are highly expressed}\newline \small P-values were calculated with Wilcoxon rank-sum tests.}\label{c3f4}}
    {\includegraphics[width=0.66\textwidth]{c3_fig4_expression}}
\end{figure}

\section{Discussion}
These findings have a number of important consequences. From a technical standpoint, they demonstrate that computational predictions of assembly order are largely accurate and biologically meaningful. Together with earlier studies on the evolutionary conservation of assembly pathways \cite{Levy2008,Marsh2013}, it is now fairly clear that in vitro pathways are similar to those in vivo - at the very least in bacteria. In eukaryotes it is known that chaperones play an extensive role in the assembly of complexes \cite{Ellis2006}, and proteins that rely on these extensively probably deviate from easily predictable assembly pathways. In archaea on the other hand, we simply don't know enough about them to make confident assertions about the regulatory mechanisms they use to aid assembly of complexes. For one thing, we have substantially less structural and sequence information available for them \cite{Mukherjee2017}, and for another, it is known that the structure of their operons differs from the canonical bacterial template \cite{Koide2009}.

% NOTE: have you used this "with these caveats" phrase before?
Bearing these limitations in mind, we can nonetheless infer some interesting biological details about the assembly process. For example, the fact that there should be such a strong correspondence between gene order and assembly order for lowly expressed complexes implies that assembly must be taking place very close to the site of translation. This is because once proteins begin to diffuse away from the polyribosome, any knowledge about the order in which they were translated is rapidly lost. In the case of less abundant protein complexes, the effect of diffusion away from each other would be particularly strong, and as a result the selective benefit they get from minimising the spatial and temporal distance between interacting subunits is probably larger than for more abundant proteins.

Taking this argument further, it seems likely that in many cases operon-encoded complexes assemble co-translationally. In the specific case of the \textit{Vibrio harveyi} heterodimeric luciferase complex, this has been demonstrated experimentally \cite{Shieh2015a}, and there is considerable evidence pointing to this being a common occurrence on a wider scale \cite{Duncan2011,Wells2015,Natan2017}.

These results demonstrate that the pressure to assemble protein complexes quickly and efficiently has been a major constraint on the evolution of gene order in bacteria. However, eukaryotes are enormously different from bacteria and archaea in ways that prevent such a system as the one described here from being possible. Most obviously, most eukaryotic species do not possess operons, and due to the existence of the nucleus, none couple transcription and translation. Indeed, the existence of operons appears to be largely decided by genome size and complexity, which on average is much greater in eukaryotes than prokaryotes \cite{Nunez2013}. And yet, given the increased size of most eukaryotic cells, along with the diversity of intracellular organelles and compartments, it seems that the need for regulatory systems that facilitate protein complex assembly should be at least equal to that of prokaryotes, if not greater. The following chapters discuss a biological phenomena that appears to be driven by such requirements.

\clearpage

\section{Supplementary information}
\vspace{40mm}
\begin{figure}[hb]
    \includegraphics{c3_figs1}
    \caption[Additional comparisons of subunits pairs encoded in the same vs. different transcriptional units]{\sffamily \textbf{Additional comparisons of subunits pairs encoded in the same vs. different transcriptional units} \\ \small (A) This figure shows the results from the same analysis as figure \ref{c3f1} B-C but using PaxDB abundance data for all organisms. The correlations between those pairs encoded in different transcriptional units is significant (p-value = 0.004), and calculated as figure \ref{c3f1}\\
    (B) Same as figure \ref{c3f1} B-C but using protein synthesis rates from ribosomal profiling data \cite{Li2014b}. P-value < \(10^{-5}\).}
    \label{c3fs1}
\end{figure}

\begin{figure}
    \includegraphics{c3_figs2}
    \caption[Relationship between gene pair proximity and likelihood of physical interaction]{\sffamily \textbf{Relationship between gene pair proximity and likelihood of physical interaction} \\ \small The four plots above display the results from comparisons of expected number of physically interacting gene pairs vs. a null model in which gene order within operons is randomly shuffled. Plots B and C relate to the \textit{nqo} operon from \textit{Thermus thermophilus}, which encodes respiratory chain complex 1. Due to its size (17 genes), it accounts for more than half of non-adjacent gene pairs in our dataset (78/148). Both within the operon itself (B), and the dataset when excluding it (C), the observed number of interacting genes is higher than expected by chance. Panel D shows the same result using binary interaction data from Y2H screens \cite{Rajagopala2014}.}
    \label{c3fs2}
\end{figure}

\begin{figure}
    \includegraphics{c3_figs3}
    \caption[Gene fusion events conserve assembly order in adjacent gene pairs]{\sffamily \textbf{Gene fusion events conserve assembly order in adjacent gene pairs} \\ \small Error bars represent Wilson 68\% binomial confidence intervals and p-values were calculated with Fisher's exact test.}
    \label{c3fs3}
\end{figure}

\begin{figure}
    \includegraphics{c3_figs4}
    \caption[Comparison of gene order, assembly order and interface size for adjacent and non-adjacent gene pairs]{\sffamily \textbf{Comparison of gene order, assembly order and interface size for adjacent and non-adjacent gene pairs} \\ \small Plots on the left describe the percentage of gene pairs for which assembly order matches gene order, split into adjacent pairs, those separated by a single intervening gene, and those separated by more than 1 gene (only genes with evolutionarily conserved order are shown.) Error bars are 68\% Wilson binomial confidence intervals. On the right, plots show the distribution of interface sizes for interacting pairs where gene order matches or doesn't match assembly order. P-values calculated with Wilcoxon rank-sum tests.}
    \label{c3fs4}
\end{figure}

\begin{figure}
    \includegraphics{c3_figs5}
    \caption[Gene order is a better predictor of assembly order than protein abundance]{\sffamily \textbf{Gene order is a better predictor of assembly order than protein abundance} \\ \small All gene pairs are those where gene order is conserved, error bars are 68\% Wilson binomial confidence intervals and p-values are Fisher's exact test.}
    \label{c3fs5}
\end{figure}

\begin{figure}
    \includegraphics{c3_figs6}
    \caption[Enrichment analysis of gene ontology terms for gene pairs in which assembly order does not match gene order]{\sffamily \textbf{Enrichment analysis of gene ontology terms for gene pairs in which assembly order does not match gene order} \\ \small Top five significant, non-redundant GO term enrichments for gene pairs in our dataset. In the above plots, `Other' refers to cases where gene order is not conserved or where there is no well-defined assembly order.GO terms were filtered for redundancy, with terms appearing together in more than 50\% of proteins in the GOA database, then only the most significant term was included in the non-redundant set. Error bars are 68\% Wilson binomial confidence intervals and p-values were calculated using approximations of Fisher's exact test, based on \(2 \times 10^{6}\) Monte Carlo iterations. The apparent enrichment for `organelle' stems from just three complexes, and is thus probably not meaningful.}
    \label{c3fs6}
\end{figure}

\begin{figure}
    \includegraphics{c3_figs7}
    \caption[Additional comparisons of protein abundance for pairs where gene order matches assembly order and vice versa]{\sffamily \textbf{Additional comparisons of protein abundance for pairs where gene order matches assembly order and vice versa} \\ \small These plots are the same as the analysis in figure \ref{c3f4}, but using abundance data from all organisms or absolute protein synthesis rates.}
    \label{c3fs7}
\end{figure}
% \printbibliography

\end{document}
