\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{../jnwthesis}
\usepackage{lipsum}
\usepackage{mhchem}
\usepackage{standalone}
\standalonetrue

% Bibliography
% \usepackage[backend=biber,style=nature,backref=true]{biblatex}
\bibliography{/Users/jonwells/Documents/bibtex/Thesis}

% Figures
% \usepackage{graphicx}
\graphicspath{{../figs/}}

\begin{document}

\chapter{Degradation kinetics of proteins are explained by assembly of protein complexes}\label{chapter:degradation}

\section{Introduction}
In the previous chapter, we saw how gene order in bacteria is optimised for assembly of protein complexes, in line with studies demonstrating that bacterial protein complex subunits are produced in proportions that correspond closely to the stoichiometry of the complex \cite{Li2014b,Burkhardt2017}. However, it is less clear that this is the case in eukaryotes, and the challenge they face in assembling protein complexes is compounded by the issues arising from their increased cell size and organisational complexity. Furthermore, as described in an influential paper by Papp, Pál and Hurst \cite{Papp2003}, there is a significant fitness cost to stoichiometric imbalances in the expression of eukaryotic protein complex subunits. The effects of this dosage sensitivity manifest in phenomena such as the rapid degradation of excess ribosomal subunits \cite{Warner1999,Sung2016}, reduced noise in expression of dosage sensitive genes \cite{Lehner2008} and an under-representation of heteromer subunits in regions with high copy number variation \cite{Schuster-Bockler2010} (CNV).

% NOTE: What proportion of the variance of mRNA/protein anti-correlation can be explained by degradation
How does the eukaryotic cell balance the need for rapid and efficient protein complex assembly with the pressure to avoid the deleterious consequences of imbalanced expression of subunits? There are several ways to consider this question, but one that I have found to be important comes from a study on protein degradation kinetics, carried out in collaboration with McShane et al \cite{McShane2016}. In this chapter, I will present the key results arising from this work, extended with some additional analyses relating to protein complex assembly and the balance hypothesis \cite{Papp2003}.

This project stems from a question posed by Matthias Selbach's lab, specifically: does the probability of a protein molecule being degraded remain constant over its lifetime? Early studies reached the conclusion that most intracellular protein degradation follows first order kinetics, implying that proteins have a fixed probability of being degraded at any moment in time \cite{Schimke1970,Goldberg1974}. However, it is not hard to imagine scenarios in which this assumption would be violated - for example, if proteins become more unstable as they accumulate damage over time, or if they accumulate modifications marking them for destruction; alternatively, nascent proteins might initially be less stable than mature ones. There is some evidence for the latter scenario from early experiments showing that many proteins (including basigin and cystic fibrosis transmembrane conductance regulator specifically) and are degraded within the first few hours following synthesis \cite{Wheatley1980,Tyler2012,Ward1994}. Similarly it has been shown that ubiquitination of nascent proteins is common, and even occurs co-translationally with some regularity \cite{Kim2011,Wang2013}.

Considerable effort has already been invested in studying protein turnover using ribosome footprint profiling and mass spectrometric methods  \cite{Ingolia2009,Ingolia2011,Doherty2009,Schwanhausser2011,Kristensen2013}, particularly since the discordance between mRNA and protein abundance first became apparent \cite{Gygi1999a,Chen2002}. However, most of the work thus far has focussed on synthesis, since degradation is challenging to measure due to the difficulties in tracking newly synthesised proteins over extended time periods. To overcome this issue, my collaborators on the project performed pulse-chase experiments using the artificial amino acid azidohomoalanine (AHA) \cite{Kiick2002,Dieterich2006}. Combining this with SILAC mass spectrometry \cite{Ong2002a}, they were able to track protein abundance changes in mammalian cells across seven time points spanning a 32 hour period. The results from this experiment yielded exciting insights into the nature of protein complex assembly in eukaryotes, lending themselves to a simple model which successfully predicts protein behaviour in aneuploid cells.

\section{Results}
\subsection{Measuring protein degradation kinetics}
Early attempts at using pulse-chase experiments to follow protein degradation were hampered by the long pulse times required to label a sufficient proportion of proteins, which often considerably overlapped with the lifetimes of the proteins being studied \cite{Larance2013}. This issue can be overcome with AHA - a bioorthogonal amino acid that has previously been used with comparatively short labelling times to track rapid changes in the proteome \cite{Eichelbaum2014}. In order to measure changes in labelled protein abundance over time (figure \ref{figure:pulsechase}A), mouse fibroblasts (NIH 3T3) were grown in heavy, medium and light SILAC medium, then pulse-labelled with AHA for one hour. The samples were then cold chased in AHA-free medium for durations specific to each of the three SILAC labels. Heavy cells were always harvested immediately after the pulse for use as the $t_{0}$ reference point for subsequent quantification. Medium and light samples were collected after cold chases of 1, 2, 4, 8, 16 and 32hrs, split across three independent replicates.

\begin{figure}
    \includegraphics{c3_fig_pulsechase}
    \caption[Quantification of protein degradation kinetics by metabolic pulse-chase labelling]{\sffamily \textbf{Quantification of protein degradation kinetics by metabolic pulse-chase labelling} \\ \small (A) Experimental setup for AHA pulse chase experiment. After a 1hr AHA pulse, each SILAC culture is harvested at different time point, with heavy cells being used as the $t_{0}$ reference point. After harvesting the cell cultures are combined, AHA proteins are enriched and then quantification of abundances carried out using tandem mass spectrometry. (B) This process was repeated in triplicate to obtain seven time points at intervals from 0 to 32 hours. Plots show mass spectra for three peptides from Flna, Ctsl1 and Bsg respectively. Relative changes in abundance are calculated from the area under different peaks. (C) Processed decay profiles for 5,247 proteins, with Flna, Ctsl1 and Bsg illustrating different possible degradation profiles. This figure is adapted from figure 1, McShane et al.\cite{McShane2016}}
    \label{figure:pulsechase}
\end{figure}

Samples were combined after harvesting to avoid introducing further batch effects and AHA proteins were enriched from this mixture on an alkyne agarose resin. The relative changes in protein abundance at each time point relative to $t_{0}$ were then quantified using LC-MS/MS. Mass spectra (MS1) for peptides from Filamin  alpha (Flna), Cathespin L1 (Ctsl1) and Basigin (Bsg) are shown in figure \ref{figure:pulsechase}B. Each of these three proteins displays a different decay profile, with Flna being very stable and decaying exponentially. Ctsl1 also decays exponentially but at a much faster rate, and ceases to be detected at time points beyond 8hrs. Finally, and most interestingly, Bsg appears to decay non-exponentially, with a rapid drop in abundance over the 8hrs hours, but very little change for the last 24hrs of the experiment. This is consistent with the earlier reports of Bsg in human cell lines undergoing rapid degradation during the first few hours of its life \cite{Tyler2012}.

Controls to were carried out to verify that AHA did not induce premature translation termination or affect protein stability. Decay profiles were normalised using a set of abundant and very stable proteins to control for differences in cell number across samples. In total 5,257 proteins were quantified (figure \ref{figure:pulsechase}), and after quality control (e.g. removing proteins with fewer than 4 data points) this was reduced to 3,605 profiles suitable for further analysis. Details on these aspects of the work can be found in the methods and supplementary material of the original research paper \cite{McShane2016}.

\subsection{Many proteins are degraded non-exponentially}
Having acquired a large set of decay profiles, we next attempted to classify proteins according to their degradation profiles. Adapting a method previously used to study mRNA degradation \cite{Deneke2013}, one- and two-state Markov models were fitted to the data for each protein (figure \ref{figure:model}A). Under the first model, there is a single transition rate (and probability) $K_{A}$ which describes the probability of a protein being degraded at any given moment in time. Under the second, there are assumed to be two states with different degradation rates, $K_{A}$ and $K_{B}$. A protein in state A can thus either be degraded or transition to state B, at which point its degradation probability will change. The one-state model therefore describes exponential degradation (ED), whilst the two-state model describes non-exponential degradation (NED).

\begin{figure}[h]
    \includegraphics{c3_fig_model}
    \caption[Non-exponentially degraded proteins are common]{\sffamily \textbf{Non-exponentially degraded proteins are common} \\ \small (A) One- and two-state Markov models used to model protein degradation profiles. $K_{X}$ describe transition probabilities between different states. (B) Markov models fitted to profiles of Flna, Ctsl1 and Bsg. Both Flna and Ctsl1 are equally well fitted by either model (i.e. $K_{A} \approx K_{B}$), as measured by the residual sum of squares (RSS), and therefore the simpler one-state model is selected. In contrast, Bsg is better fit by a two-state model, indicated by the favourable RSS and high AIC. (C) Under conservative thresholds for AIC and $\Delta$-score 10\% of proteins are degraded non-exponentially. Adapted from figure 2, McShane et al.\cite{McShane2016}}
    \label{figure:model}
\end{figure}

To determine which was the better fit for each protein, we used the Akaike information criterion \cite{Akaike1974} (AIC), which imposes a penalty for additional parameters and therefore gives preference to the one-state model if both explain the data similarly well. However, since AIC describes goodness of fit, but not the extent to which non-exponential degradation occurs. For example, a protein may be better fit by the two state model but only deviate weakly from a single degradation rate. To include this feature in classifications, a measure was developed ($\Delta$-score) to assess the degree to which a pre-decided time point deviates from that expected under an exponential distribution. Specifically, the distance of the datapoint at $t_{8}$ from the line fit by the one-state model (see methods in the original paper for details - appendix \ref{appendix:published}). ED proteins were then classified as those which had a 2-state $AIC < 0.2$ and $\lvert \Delta \rvert < 0.15$, NED proteins as those with $AIC > 0.8$ and $\lvert \Delta \rvert > 0.15$, with proteins not meeting either criteria being undefined. Using the example cases of Flna, Ctsl1 and Bsg (figure \ref{figure:model}B), we found that the first two are best fit by the one-state model, whereas the two-state model is preferred for Bsg, as one would expect from a visual assessment of the plots.

When we applied this classification scheme to the set of 3,292 proteins whose decay profiles passed quality control and for which $\Delta$-scores could be calculated, we found that the majority of proteins (49\%) were degraded exponentially, consistent with traditional models (figure \ref{figure:model}C). Strikingly however, some 10\% of proteins are best fit by a non-exponential model of degradation, despite the conservative nature of our classifier. A particularly unexpected finding was that in all NED cases, $K_{A}$ was greater than $K_{B}$, indicating that these proteins are always initially rapidly degraded, before becoming more stable later in their lives. This suggests that, at least in the 32 hour time period we used, age- or damage-related destabilisation is rare, if it occurs at all.

\subsection{NED proteins are degraded via the ubiquitin-proteasome system}
There are two cellular systems concerned with degradation of proteins - the ubiquitin-proteasome system and the lysosomal system. To determine which of these was primarily responsible for degradation in the case of NED proteins, my collaborators inhibited each in turn using MG132, or wortmannin in combination with bafilomycin A1, respectively. DMSO was used a as a carrier control, and deviation from behaviour versus this control was measured by the change in $\Delta$-score. Of these two treatments, MG132 lead to a marked decrease in NED character, whilst wortmannin in combination with bafilomycin A1 had no appreciable effect (figure \ref{figure:proteasome}). This indicates that initial rapid degradation of NED proteins is due to the ubiquitin-proteasome pathway, whilst the lysosomal system plays a negligible role.

\begin{figure}[h]
    \includegraphics{c3_fig_proteasome}
    \caption[NED proteins are degraded via the ubiquitin-proteasome system]{\sffamily \textbf{NED proteins are degraded via the ubiquitin-proteasome system} \\ \small (A) Example case showing hypothetical decrease in $\Delta$-score caused by treatment at 4 and 8hr time points. (B-C) Results from treatment with MG132 or wortmannin in combination with bafilomycin A1, demonstrating significant reductions in NED protein $\Delta$-score caused by the former. P-values calculated with one-sided Wilcoxon rank-sum tests; ***p < 0.0001. (D) Effect of MG132 on degradation of all proteins versus DMSO control. (E) Control demonstrating inhibition of lysosomal degradation with the wortmannin-bafilomycin A1 combination (labelled `Autophagy'). The accumulation of the autophagy marker LC3-II indicates that lysosomal degradation has been successfully blocked. Adapted from figure 4, McShane et al.\cite{McShane2016}}
    \label{figure:proteasome}
\end{figure}

\subsection{NED proteins are enriched in heteromeric protein complexes}
Based on analysis of a previously published set on manually curated protein complexes \cite{Ori2016}, my collaborators had found that NED proteins appeared to be over-represented in protein complexes. Building on this finding, I mapped proteins from the degradation dataset to protein structures from the PDB, revealing that approximately 70\% of NED proteins are members of heteromeric protein complexes\footnote{Though this may be a conservative estimate since protein complexes, particularly larger ones, are still somewhat under-sampled in the PDB relative to monomeric or homomeric structures. Intriguingly, the number of novel homomeric and monomeric proteins released per year appears to be slowing down \cite{Perica2012a,Marsh2014}. However, this is probably a result of increased interest in solving heteromeric structures caused by technological developments discussed in chapter \ref{chapter:intro}, rather than a sign of saturation of monomeric structure space.}, which is a significant enrichment compared to either monomers or homomers (figure \ref{figure:nedcomplex}A). To exclude the possibility that this trend was driven exclusively by the ribosome (subunits of which are prevalent in our structural dataset), I repeated this analysis with ribosomes filtered from the dataset and obtained the same trend, with comparable statistical significance (figure \ref{suppfigure:ribocontrol}A).

Several independent studies have suggested that assembly of protein complexes stabilises the proteins involved \cite{Goldberg2003,Malinverni2006,Toyama2013}. These earlier studies are therefore supported by our findings that, firstly, NED proteins in our dataset exclusively transition to more stable states with age, and secondly, are enriched in heteromeric protein complexes.

% This finding was replicated using CORUM, a manually curated database of experimentally validated protein complexes (non-structural and excluding high-throughput data) compiled from the literature \cite{Ruepp2010}.

\begin{figure}
    \includegraphics{c3_fig_nedcomplex}
    \caption[NED proteins are enriched in heteromeric protein complexes]{\sffamily \textbf{NED proteins are enriched in heteromeric protein complexes} \\ \small (A) NED proteins are proportionally much more common in heteromers than either monomers or homomers. P-values were calculated with Fisher's exact test, comparing the number of heteromeric subunits to monomeric or homomeric. (B) Considering only heteromeric protein complexes, NED proteins are more evenly distributed across the range of sizes, as measured by the number of unique subunits in each complex. P-value calculated using a modified Kolmogorov-Smirnov test to account for the discrete distribution of subunit counts - see R package `dgof' \cite{Arnold2011}. (C) NED proteins tend to form larger interfaces, including when controlling for protein complex size. (D) NED proteins also tend to assemble earlier. Assembly order is predicted using the method described in Marsh et al. \cite{Marsh2013}, and normalised between zero (early) and one (late). (E) Mean coexpression scores were calculated for each protein, based on all pairwise correlations with other proteins in the complex of interest. Expression of NED proteins are significantly more likely to be closely correlated with other subunits, compared against ED. P-values in panels C-E calculated with Wilcoxon rank-sum tests. Adapted from figure 5, McShane et al.\cite{McShane2016}}
    \label{figure:nedcomplex}
\end{figure}

Further supporting these studies, I observed a highly significant tendency for NED proteins to be found in large heteromers (measured in terms of number of unique subunits) compared to ED proteins (figure \ref{figure:nedcomplex}B). This is a reasonable finding if one assumes that as the size of complexes increases, so too will the proportion of subunits which are protected from degradation. Again, I repeated this analysis excluding ribosomes, and the enrichment of NED proteins in large complexes remained largely unchanged (figure \ref{suppfigure:ribocontrol}B).

Within individual protein complexes, NED proteins also tend to form larger interfaces than ED. Importantly, this trend holds when controlling for number of unique subunits (figure \ref{figure:nedcomplex}C), since larger complexes typically contain larger interfaces and from the last analysis we know that NED proteins are also enriched in these complexes. Intriguingly, despite the fact that NED proteins are not enriched in homomers (complexes with one unique subunit), there is nonetheless a significant tendency for homomeric NED proteins to form larger interfaces than ED.

Since there is a strong relationship between interface size and protein complex assembly order \cite{Marsh2013}, it seemed likely that NED proteins would assemble earlier. Assembly order predictions were generated for the complexes in our structural dataset, and then normalised between 0 and 1, with lower numbers indicating earlier assembly. Since the assembly order of small complexes is relatively uninformative, I restricted the analysis to protein complexes with more than five subunits. Comparing the assembly position of NED and ED subunits, there is a weak tendency for the latter to assemble later, with undefined subunits somewhere between the two (figure \ref{figure:nedcomplex}D).

Finally, I looked at the tendency for NED and ED proteins to be coexpressed with other members of the protein complex. To assign a single coexpression score to each protein complex subunit, I calculated the mean pairwise mRNA coexpression score between that subunit and all others in the complex. Comparing the two degradation classes, the expression of NED proteins is more tightly coupled to the rest of the complex (figure \ref{figure:nedcomplex}E).

\subsection{Results are replicable across species and protein complex datasets}
In order to ensure that these results were not specific to the mouse cell line being used (NIH 3T3), which has previously been shown to have an abnormal karyotype \cite{Leibiger2013}, my collaborators repeated the AHA-SILAC experiments using a human cell line (RPE1). This cell line was shown to have a normal karyotype, with the exception of partial trisomies of chromosomes ten and twelve. Their analyses of this data revealed that most proteins retain ED or NED character in human/mouse orthologues, and that $\Delta$-score profiles of human versus mouse proteins are correlated (Pearson’s correlation coefficient, $r = 0.39$, $p < 2.2e^{-16}$). I then repeated the structural analyses described in figure \ref{figure:nedcomplex} using the human dataset; this produced highly similar results (figure \ref{suppfigure:human}), albeit with weaker statistical significance - probably as a result of the lower quality of the human data, with 49\% of proteins passing quality control in this dataset compared to >60\% in the mouse data.

Using non-structural data from CORUM  \cite{Ruepp2009} I repeated the coexpression analysis using mouse (figure \ref{figure:corum}A) and human data (figure \ref{figure:humancorum}) datasets. In both cases there was a significant tendency for NED proteins to be more highly coexpressed. Since proteins from larger complexes such as the ribosome or proteasome tend to be highly correlated in their expression, I also split proteins into bins based on the number of unique subunits in the complex they were associated with (figure \ref{figure:corum}B). This showed that whilst there is a (not unexpected) tendency for NED coexpression to be stronger in larger complexes, overall the trend is not driven exclusively my complex size.

A final interesting analysis comes from looking at a mass-spectrometric dataset in which protein complexes were identified and quantified according to their stoichiometries and absolute abundances \cite{Hein2015}. This approach allowed them to distinguish between `core' protein complex subunits, which were identified by their closely matched stoichiometries both within complexes and across the entire cell. If NED proteins are indeed core subunits, as our analyses seem to suggest, then we would expect them to be enriched in the core set from the mass-spectrometric dataset. This turns out to be the case, with human NED proteins being significantly more likely to be identified as core interactors compared to ED (Fisher's exact test, odds ratio = 3.0, $p < 2.2e^{-16}$).

\begin{figure}
    \includegraphics[width=\textwidth]{c3_fig_corum}
    \caption[Increased NED protein coexpression is not unique to structural data]{\sffamily \textbf{Increased NED protein coexpression is not unique to structural data} \\ \small (A) Replicate of figure \ref{figure:nedcomplex}E using mouse data and experimentally validated protein complexes from CORUM \cite{Ruepp2009}. The bimodal distribution of NED subunits is likely due to the strong influence of the ribosome, which is tightly regulated and contributes a substantial number of all proteins in the dataset. (B) This trend holds when controlling for the size of protein complexes, measured in terms of number of unique subunits. P-values calculated with Wilcoxon rank-sum tests.}
    \label{figure:corum}
\end{figure}

\subsection{Protein complex assembly explains degradation kinetics}
Collectively, these observations suggest that NED proteins tend to be core subunits within protein complexes, whereas ED proteins are more likely to be monomeric/homomeric or participate as peripheral subunits within heteromers. A simple model that would explain why core members of subunits would display non-exponetial degradation kinetics is shown in figure \ref{figure:nedabundance}A. If core subunits are more abundant than peripheral subunits, then there would always a fraction of proteins unable to assemble into complexes, and thus subject to rapid degradation. My collaborators estimated absolute protein abundances after the AHA pulse using iBAQ \cite{Schwanhausser2011}, and mapped these to the set of protein complexes they used previously \cite{Ori2016}. Normalising abundances relative to the mean of each complex, we found that NED proteins within complexes were significantly more abundant than ED (figure \ref{figure:nedabundance}B), consistent with our model. In addition, the second-state degradation rates of NED proteins were more similar to those of ED within complexes.

\begin{figure}[h]
\fcapsideright
    {\caption[NED proteins are produced in excess in heteromeric complexes]{\sffamily\textbf{NED proteins are produced in excess in heteromeric complexes} \\ \small (A) Model showing how overproduction and subsequent degradation of excess NED subunits facilitates protein complex assembly. (B) NED subunits are significantly more abundant per complex than ED proteins. $\log{2}$ abundance fold change of each subunits calculated against the mean abundance of each complex. P-value calculated with Wilcoxon rank-sum test. Adapted from figure 5, McShane et al. \cite{McShane2016}}\label{figure:nedabundance}}
    {\includegraphics[width=0.6\textwidth]{c3_fig_abundance}}
\end{figure}

The finding that NED proteins (core protein complex subunits) tend to be over-produced relative to ED (more peripheral) is somewhat paradoxical in light of the dosage balance hypothesis described by Papp et al \cite{Papp2003}. This hypothesis states that changes in gene expression that affect the stoichiometric imbalance should be deleterious, and indeed there is much evidence to support this; however much of it relates to large changes in gene expression caused by copy number variants. If excess heteromeric subunits are rapidly degraded, as demonstrated by the results presented in this chapter, then this suggests that smaller fluctuations in expression might be better tolerated by heteromers than monomeric or homomeric proteins - more specifically, increases in expression should be better tolerated than decreases, since these can be degraded. To test this idea, I mapped eQTL data from the Genotype-Tissue Expression \cite{Brown2016} (GTEx) project onto a set of human protein complexes; if correct, then heteromers should on average have more eQTLs per gene. Separating these out by quaternary structure category revealed that, contrary to this hypothesis, the number of significant eQTLs per gene is lower for heteromeric subunits, in accordance with results supporting the balance hypothesis (figure \ref{figure:eqtls}). Moreover, there are not meaningful differences between the number of upregulating and downregulating eQTLs across different quaternary structure types.

\begin{figure}[h]
\fcapsideright
    {\caption[eQTLs are less frequent for heteromeric proteins]{\sffamily\textbf{eQTLs are less frequent for heteromeric proteins} \\ \small If rapid degradation of excess heteromer subunits acts to increase robustness in the expression of protein complexes, then we would expect eQTLs to be better tolerated in these proteins. However, this does not appear to be the case, as evidenced by the lower frequency of eQTLS per gene in heteromers. P-values calculated with Wilcoxon rank-sum tests.}\label{figure:eqtls}}
    {\includegraphics[width=0.6\textwidth]{c3_fig_eqtl}}
\end{figure}

\section{Discussion}

This study of protein degradation kinetics revealed that many proteins are degraded non-exponentially, indicating that examples previously reported in the literature are in fact part of a widespread phenomenon. To explain these observations, we proposed a model in which protein complex assembly stabilises those proteins involved, with excess subunits being degraded. As we shall see in the next chapter, this model successfully explains protein attenuation in aneuploid cells. In addition, the findings support the idea that many heteromers, particularly larger ones, possess sets of core subunits whose structural and behavioural properties differ markedly from peripheral or non-essential subunits. However, the work also raises questions; for example, what are the mechanisms that explain why assembly into complexes stabilises proteins, and why are eukaryotic protein subunits not produced at levels corresponding to their stoichiometry, as appears to be the case in bacteria \cite{Li2014b}?

There are a few mechanisms could explain why proteins are stabilised upon binding. The simplest is that there is a safety-in-numbers effect, whereby proteins in large complexes that bury more surface area are less accessible to the proteasome. This is certainly consistent with the tendency of NED proteins to have larger interfaces and assemble earlier. However, this alone does not explain why NED subunits are initially degraded faster than ED subunits within complexes. One factor that is probably important is ubiquitination - it would be interesting to see if there is enrichment for ubiquitination sites in NED proteins or protein complex subunits in general, some evidence seems to point to \cite{Chen2014}. If this is the case, are ubiquitination sites enriched or depleted in protein interfaces, and can ubiquitinated proteins still assemble into functional complexes?

% NOTE: Improve, tighten up and shorten a little
The published work this chapter derives from discusses a number of possible reasons why NED proteins might be overexpressed relative to ED. One interesting idea is that overexpression of core subunits represents a simple mechanism for controlling levels of the holocomplex. If the central NED subunits are constitutively overexpressed as a single, correlated group relative to late-assembling ED subunits, then the levels of the fully assembled complex can be modulated by that of the ED subunits alone. This fits with the observation that expression of ED subunits is often decoupled from that of the rest of the complex. On the other hand, since most heteromeric subunits are NED, the finding that eQTLs are still less common in heteromers suggests that NED proteins are not necessarily any more robust to changes in expression.

A simpler alternative is that overexpression of early-assembling core subunits is a necessity in ensuring efficient assembly, since the encounter of interacting proteins is concentration dependent, and eukaryotes do not benefit from the high local concentration of subunits produced by encoding heteromers in operons. This scenario goes some way to explaining why eukaryotic subunits are not expressed at stoichiometric ratios, and does not contradict the balance hypothesis, since proteins may be expressed non-stoichiometrically, but nonetheless at tightly controlled levels.

In summary, this study shows that non-exponential degradation of proteins is a commonplace. This process is driven by the ubiquitin-proteasome system and the inherent instability of core heteromeric subunits. It is important to note that whilst NED character is conserved across species, it is not an inherent property of the proteins themselves, but rather an emergent phenomena caused by the requirements of protein complex assembly in eukaryotes. This can be shown by the fact that NED character decreases in response to repressing the proteasome, and the way in which proteins behave in aneuploid cells.

% \printbibliography

\end{document}
